<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Elasticizing tissues for reversible shape transformation and accelerated molecular labeling | Nature Methods</title>

    
        
<link rel="preload" href=/static/fonts/Lora-Regular.8861b0072d.woff2 as="font" type="font/woff2" crossorigin>


<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=2.5,user-scalable=yes"/>



<script>
    (function(e){var t=e.documentElement,n=e.implementation;t.className+=' js';if(n&&n.hasFeature('http://www.w3.org/TR/SVG11/feature#Image','1.1')){t.className+=' svg'}})(document)
</script>

<script data-test="dataLayer">
    dataLayer = [{"content":{"category":{"contentType":"brief communication","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"fluorescence-imaging;histology;neuroscience","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Methods","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Brief Communication"}},"article":{"doi":"10.1038/s41592-020-0823-y"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Taeyun Ku","Webster Guan","Nicholas B. Evans","Chang Ho Sohn","Alexandre Albanese","Joon-Goon Kim","Matthew P. Frosch","Kwanghun Chung"],"publishedAt":1589760000,"publishedAtString":"2020-05-18","title":"Elasticizing tissues for reversible shape transformation and accelerated molecular labeling","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"nmeth","title":"nature methods","volume":null,"issue":null},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":false}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"ab_test_pandemic_bypass_paywall","active":false},{"name":"ab_test_best_available_version","active":false},{"name":"ab_test_magazine_article_native_ads","active":false}]},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true}];
</script>
<script src="//cdn.cookielaw.org/consent/792a5a21-3d99-4432-8f70-957d389bf2b9.js" async=""></script>
<script id="js-position0">
    window.abTestSharedArticleRenderer = true;
    window.idpVerifyPrefix = 'https://verify.nature.com';
    window.ra21Host = 'https://wayf.springernature.com';
</script>


    <script>
        (function() {
            window.onerror = function(message, url, line, column, error) {
                if (window.Raven) {
                    window.Raven.captureException(error);
                } else {
                    var s = document.createElement('script');
                    s.src = 'https://cdn.ravenjs.com/3.27.0/raven.min.js';
                    s.crossorigin = 'anonymous';
                    s.defer = 'defer';
                    s.onload = function() {
                        window.Raven.config('https://bddad754e8264fc7b4d73878b697b599@sentry.i-ris.io/39', {
                            whitelistUrls: [
                                'www.nature.com/'
                            ]
                        }).install();
                        window.Raven.captureException(error);
                    };
                    document.body.appendChild(s);
                }
            };

            if (window.matchMedia && window.matchMedia('only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)').matches) {
                var index = 0;
                var scripts = [
                    {src: '/static/js/nature.7eb33788ae.js', test: 'nature-js'},
                    {src: '/static/js/nature-es6-bundle.a345c5a891.js', test: 'nature-es6-js'},
                    {src: '/static/js/shared-es6-bundle.076f9d1226.js', test: 'shared-es6-js'}
                    
                ];
                var initOnceWhenReady = function() {
                    if (window.initEs5 && window.initEs6) {
                        window.initEs5();
                        window.initEs6();
                        window.initEs5 = null;
                        window.initEs6 = null;
                    }
                };
                var createScript = function(src, test, beforeLoad) {
                    var s = document.createElement('script');
                    s.id = 'js-position' + (index + 1);
                    s.setAttribute('data-test', test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function() {
                            if (document.readyState !== 'loading') {
                                initOnceWhenReady();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = src;
                    return s;
                };
                var insert = function(s) {
                    document.getElementById('js-position' + index).insertAdjacentElement('afterend', s);
                    ++index;
                };

                scripts.forEach(function (script) {
                    insert(createScript(script.src, script.test, true));
                });

                document.addEventListener('DOMContentLoaded', function() {
                    initOnceWhenReady();

                    var conditionalScripts = [
                        {match: 'div[data-pan-container]', src: '/static/js/pan-zoom.742a102a55.js', test: 'pan-zoom-js'},
                        {match: 'math,span.mathjax-tex', src: '/static/js/math.acd6b1bcfe.js', test: 'math-js'}
                    ];
                    conditionalScripts.filter(function(script) {
                        return !!document.querySelector(script.match);
                    }).forEach(function(script) {
                        insert(createScript(script.src, script.test));
                    });
                }, false);
            }
        })();
    </script>




<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon.f39cb19454.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32.3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16.951651ab72.png>
<link rel="manifest" href=/static/manifest.1a481c42b1.json>
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab.69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.62367f778b.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.e35b3b052c.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">




    
     
        
        
            <link id="mustard" rel="stylesheet" type="text/css" href="/static/css/article.f4267e4b8b.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)" />
        
        <link rel="stylesheet" type="text/css" href="/static/css/article-print.64c3362d02.css" media="print">
    



    
        
            <style>
                .header-logo img {
                    max-width: 280px;
                }

                 .background-brand-primary {
                      background-color: #617693;
                      color: #fff;
                 }

                 .background-brand-secondary {
                      background-color: #3b5477;
                      color: #fff;
                 }

                 .background-brand-secondary-pdf {
                      background-color: #036599;
                      background-size: 22px auto;
                      background-position: 80% 50%;
                      color: #fff;
                 }

                 .background-brand-secondary-pdf a {
                      color: #fff;
                 }

                 .background-brand-tertiary {
                      background-color: #dbdcdf;
                      color: #222;
                 }

                 .background-gradient-brand-primary {
                      background-image: linear-gradient(to right, #617693, #617693 50%, transparent 50%);
                 }

                 .background-gradient-brand-secondary-tertiary {
                      background-image: linear-gradient(to right, #3b5477, #3b5477 50%, #dbdcdf 50%);
                 }

                 .background-gradient-brand-gray {
                      background-image: linear-gradient(to right, #e0e0e0, #e0e0e0 50%, transparent 50%);
                 }

                 .background-gradient-brand-gray-light {
                      background-image: linear-gradient(to right, #eee, #eee 50%, transparent 50%);
                 }

                 
                      .background-brand-primary a {
                          color: #fff;
                      }
                 

                 
                      .background-brand-secondary a {
                          color: #fff;
                      }
                 

                 

                 .nav-border {
                      border-bottom: 4px solid #3b5477;
                 }

                 /* less than or equal to 875px */
                 @media only screen and (max-width: 54.688em) {
                      .banner {
                          background-color: #617693;
                      }

                      .nav-border .menu-cell li {
                          border-bottom: 1px solid #3b5477;
                      }
                 }

                 .sticky-header .inner-banner,
                 .header-primary-color {
                      background-color: #617693;
                 }

                 
                     .menu-button,
                     .menu-button-clone,
                     .header-submit-button {
                         border: 1px solid #fff;
                         border: 1px solid rgba(255, 255, 255, 0.75);
                         color: #fff;
                     }
                     .header-top-bar {
                         border-bottom: 1px solid #fff;
                         border-bottom: 1px solid rgba(255, 255, 255, 0.75);
                     }

                     .icon-rotate.tools-menu-button-icon:after,
                     .menu-button-icon:after {
                          background-image: url("/static/images/icons/icon-arrow-down-12x7-white.e10aedb54f.svg"), none;
                     }

                     .nature-research-logo {
                          background-image: url("/static/images/product-logos/nature-research-logo-white.a9028dbd21.svg"), none;
                     }
                     .header-search-button {
                          padding-right: 25px;
                          background-image: url("/static/images/icons/icon-search-24x24-white.7b3ba09d04.svg"), none;
                     }
                 
             </style>
         
    


<script>(function(w,d){if(w.matchMedia && w.matchMedia(d.getElementById('mustard').media).matches){d.documentElement.className=d.documentElement.className.replace(/\s*grade-c/, "")}})(window,document)</script>


<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-NWDMT9Q');</script>




<meta name="robots" content="noarchive">
<meta name="access" content="Yes">
<meta name="WT.cg_s" content="Article"/>
<meta name="WT.z_bandiera_abtest" content="a"/>
<meta name="WT.page_categorisation" content="Article_HTML"/>

    <meta name="WT.template" content="oscar"/>
    <meta name="WT.z_cg_type" content="Nature Research Journals"/>
    <meta name="WT.cg_n" content="Nature Methods"/>
    <meta name="dc.rights" content="©2020 Macmillan Publishers Limited. All Rights Reserved."/>
    <meta name="prism.issn" content="1548-7105"/>


<link rel="search" href="http://www.nature.com/search">
<link rel="search" href="http://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="http://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">


    
    

    <meta name="journal_id" content="41592"/>

    <meta name="dc.title" content="Elasticizing tissues for reversible shape transformation and accelerated molecular labeling"/>

    <meta name="dc.source" content="Nature Methods 2020"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="Nature Publishing Group"/>

    <meta name="dc.date" content="2020-05-18"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>

    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>

    <meta name="dc.description" content="We developed entangled link-augmented stretchable tissue-hydrogel (ELAST), a technology that transforms tissues into elastic hydrogels to enhance macromolecular accessibility and mechanical stability simultaneously. ELASTicized tissues are highly stretchable and compressible, which enables reversible shape transformation and faster delivery of probes into intact tissue specimens via mechanical thinning. This universal platform may facilitate rapid and scalable molecular phenotyping of large-scale biological systems, such as human organs. The ELAST technology renders large tissue samples elastic through embedding in hydrogel and facilitates sample handling and rapid molecular labeling."/>

    <meta name="prism.issn" content="1548-7105"/>

    <meta name="prism.publicationName" content="Nature Methods"/>

    <meta name="prism.publicationDate" content="2020-05-18"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="1"/>

    <meta name="prism.endingPage" content="5"/>

    <meta name="prism.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>

    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>

    <meta name="prism.url" content="https://www.nature.com/articles/s41592-020-0823-y"/>

    <meta name="prism.doi" content="doi:10.1038/s41592-020-0823-y"/>

    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41592-020-0823-y.pdf"/>

    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41592-020-0823-y"/>

    <meta name="citation_journal_title" content="Nature Methods"/>

    <meta name="citation_journal_abbrev" content="Nat Methods"/>

    <meta name="citation_publisher" content="Nature Publishing Group"/>

    <meta name="citation_issn" content="1548-7105"/>

    <meta name="citation_title" content="Elasticizing tissues for reversible shape transformation and accelerated molecular labeling"/>

    <meta name="citation_online_date" content="2020/05/18"/>

    <meta name="citation_firstpage" content="1"/>

    <meta name="citation_lastpage" content="5"/>

    <meta name="citation_article_type" content="Brief Communication"/>

    <meta name="citation_language" content="en"/>

    <meta name="dc.identifier" content="doi:10.1038/s41592-020-0823-y"/>

    <meta name="DOI" content="10.1038/s41592-020-0823-y"/>

    <meta name="citation_doi" content="10.1038/s41592-020-0823-y"/>

    <meta name="description" content="We developed entangled link-augmented stretchable tissue-hydrogel (ELAST), a technology that transforms tissues into elastic hydrogels to enhance macromolecular accessibility and mechanical stability simultaneously. ELASTicized tissues are highly stretchable and compressible, which enables reversible shape transformation and faster delivery of probes into intact tissue specimens via mechanical thinning. This universal platform may facilitate rapid and scalable molecular phenotyping of large-scale biological systems, such as human organs. The ELAST technology renders large tissue samples elastic through embedding in hydrogel and facilitates sample handling and rapid molecular labeling."/>

    <meta name="dc.creator" content="Taeyun Ku"/>

    <meta name="dc.creator" content="Webster Guan"/>

    <meta name="dc.creator" content="Nicholas B. Evans"/>

    <meta name="dc.creator" content="Chang Ho Sohn"/>

    <meta name="dc.creator" content="Alexandre Albanese"/>

    <meta name="dc.creator" content="Joon-Goon Kim"/>

    <meta name="dc.creator" content="Matthew P. Frosch"/>

    <meta name="dc.creator" content="Kwanghun Chung"/>

    <meta name="dc.subject" content="Fluorescence imaging"/>

    <meta name="dc.subject" content="Histology"/>

    <meta name="dc.subject" content="Neuroscience"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Structural and molecular interrogation of intact biological systems; citation_author=K Chung; citation_volume=497; citation_publication_date=2013; citation_pages=332-337; citation_doi=10.1038/nature12107; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Expansion microscopy; citation_author=F Chen, PW Tillberg, ES Boyden; citation_volume=347; citation_publication_date=2015; citation_pages=543-548; citation_doi=10.1126/science.1260088; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Multiplexed intact-tissue transcriptional analysis at cellular resolution; citation_author=EL Sylwestrak, P Rajasethupathy, MA Wright, A Jaffe, K Deisseroth; citation_volume=164; citation_publication_date=2016; citation_pages=792-804; citation_doi=10.1016/j.cell.2016.01.038; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues; citation_author=T Ku; citation_volume=34; citation_publication_date=2016; citation_pages=973-981; citation_doi=10.1038/nbt.3641; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Three-dimensional intact-tissue sequencing of single-cell transcriptional states; citation_author=X Wang; citation_volume=361; citation_publication_date=2018; citation_pages=eaat5691; citation_doi=10.1126/science.aat5691; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Tissue clearing and its applications in neuroscience; citation_author=HR Ueda; citation_volume=21; citation_publication_date=2020; citation_pages=61-79; citation_doi=10.1038/s41583-019-0250-1; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Nanoscale imaging of RNA with expansion microscopy; citation_author=F Chen; citation_volume=13; citation_publication_date=2016; citation_pages=679-684; citation_doi=10.1038/nmeth.3899; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Adv. Sci.; citation_title=Scalable and isotropic expansion of tissues with simply tunable expansion ratio; citation_author=H-E Park; citation_volume=6; citation_publication_date=2019; citation_pages=1901673; citation_doi=10.1002/advs.201901673; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Advances in engineering hydrogels; citation_author=YS Zhang, A Khademhosseini; citation_volume=356; citation_publication_date=2017; citation_pages=eaaf3627; citation_doi=10.1126/science.aaf3627; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Adv. Mater.; citation_title=The polyrotaxane gel: a topological gel by figure-of-eight cross-links; citation_author=Y Okumura, K Ito; citation_volume=13; citation_publication_date=2001; citation_pages=485-487; citation_doi=10.1002/1521-4095(200104)13:7&lt;485::AID-ADMA485&gt;3.0.CO;2-T; citation_id=CR10"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Quantifying the impact of molecular defects on polymer network elasticity; citation_author=MJ Zhong, R Wang, K Kawamoto, BD Olsen, JA Johnson; citation_volume=353; citation_publication_date=2016; citation_pages=1264-1268; citation_doi=10.1126/science.aag0184; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Adv. Mater.; citation_title=Double-network hydrogels with extremely high mechanical strength; citation_author=JP Gong, Y Katsuyama, T Kurokawa, Y Osada; citation_volume=15; citation_publication_date=2003; citation_pages=1155-1158; citation_doi=10.1002/adma.200304907; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Highly stretchable and tough hydrogels; citation_author=J-Y Sun; citation_volume=489; citation_publication_date=2012; citation_pages=133-136; citation_doi=10.1038/nature11409; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Materials both tough and soft; citation_author=JP Gong; citation_volume=344; citation_publication_date=2014; citation_pages=161-162; citation_doi=10.1126/science.1252389; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Polymer; citation_title=Elasticity of entangled networks; citation_author=RC Ball, M Doi, SF Edwards, M Warner; citation_volume=22; citation_publication_date=1981; citation_pages=1010-1018; citation_doi=10.1016/0032-3861(81)90284-6; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Scale: a chemical approach for fluorescence imaging and reconstruction of transparent mouse brain; citation_author=H Hama; citation_volume=14; citation_publication_date=2011; citation_pages=1481-1488; citation_doi=10.1038/nn.2928; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=SeeDB: a simple and morphology-preserving optical clearing agent for neuronal circuit reconstruction; citation_author=M-T Ke, S Fujimoto, T Imai; citation_volume=16; citation_publication_date=2013; citation_pages=1154-1161; citation_doi=10.1038/nn.3447; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Shrinkage-mediated imaging of entire organs and organisms using uDISCO; citation_author=C Pan; citation_volume=13; citation_publication_date=2016; citation_pages=859-867; citation_doi=10.1038/nmeth.3964; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A three-dimensional single-cell-resolution whole-brain atlas using CUBIC-X expansion microscopy and tissue clearing; citation_author=TC Murakami; citation_volume=21; citation_publication_date=2018; citation_pages=625-637; citation_doi=10.1038/s41593-018-0109-1; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Simple, scalable proteomic imaging for high-dimensional profiling of intact systems; citation_author=E Murray; citation_volume=163; citation_publication_date=2015; citation_pages=1500-1514; citation_doi=10.1016/j.cell.2015.11.025; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Cellular and molecular probing of intact human organs; citation_author=S Zhao; citation_volume=180; citation_publication_date=2020; citation_pages=796-812; citation_doi=10.1016/j.cell.2020.01.030; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Cell Rep.; citation_title=Whole-body profiling of cancer metastasis with single-cell resolution; citation_author=SI Kubota; citation_volume=20; citation_publication_date=2017; citation_pages=236-250; citation_doi=10.1016/j.celrep.2017.06.010; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Protection of tissue physicochemical properties using polyfunctional crosslinkers; citation_author=Y-G Park; citation_volume=37; citation_publication_date=2019; citation_pages=73-83; citation_doi=10.1038/nbt.4281; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=Generation of cerebral organoids from human pluripotent stem cells; citation_author=MA Lancaster, JA Knoblich; citation_volume=9; citation_publication_date=2014; citation_pages=2329-2340; citation_doi=10.1038/nprot.2014.158; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Mol. Psychiatr.; citation_title=MeCP2-regulated miRNAs control early human neurogenesis through differential effects on ERK and AKT signaling; citation_author=N Mellios; citation_volume=23; citation_publication_date=2018; citation_pages=1051-1065; citation_doi=10.1038/mp.2017.86; citation_id=CR25"/>

    <meta name="citation_author" content="Taeyun Ku"/>

    <meta name="citation_author_institution" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, USA"/>

    <meta name="citation_author_institution" content="Picower Institute for Learning and Memory, MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Graduate School of Medical Science and Engineering, KAIST, Daejeon, Republic of Korea"/>

    <meta name="citation_author" content="Webster Guan"/>

    <meta name="citation_author_institution" content="Department of Chemical Engineering, MIT, Cambridge, USA"/>

    <meta name="citation_author" content="Nicholas B. Evans"/>

    <meta name="citation_author_institution" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, USA"/>

    <meta name="citation_author_institution" content="Picower Institute for Learning and Memory, MIT, Cambridge, USA"/>

    <meta name="citation_author" content="Chang Ho Sohn"/>

    <meta name="citation_author_institution" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, USA"/>

    <meta name="citation_author_institution" content="Picower Institute for Learning and Memory, MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea"/>

    <meta name="citation_author_institution" content="Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea"/>

    <meta name="citation_author" content="Alexandre Albanese"/>

    <meta name="citation_author_institution" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, USA"/>

    <meta name="citation_author" content="Joon-Goon Kim"/>

    <meta name="citation_author_institution" content="Graduate School of Medical Science and Engineering, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea"/>

    <meta name="citation_author" content="Matthew P. Frosch"/>

    <meta name="citation_author_institution" content="C.S. Kubik Laboratory for Neuropathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA"/>

    <meta name="citation_author" content="Kwanghun Chung"/>

    <meta name="citation_author_institution" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, USA"/>

    <meta name="citation_author_institution" content="Picower Institute for Learning and Memory, MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Department of Chemical Engineering, MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Department of Brain and Cognitive Sciences, MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Broad Institute of Harvard University and MIT, Cambridge, USA"/>

    <meta name="citation_author_institution" content="Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea"/>

    <meta name="citation_author_institution" content="Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea"/>

    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>

    <meta name="twitter:card" content="summary"/>

    <meta name="twitter:title" content="Elasticizing tissues for reversible shape transformation and accelerat"/>

    <meta name="twitter:site" content="@naturemethods"/>

    <meta name="twitter:description" content="The ELAST technology renders large tissue samples elastic through embedding in hydrogel and facilitates sample handling and rapid molecular labeling."/>

    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig1_HTML.png"/>

    <meta name="twitter:image:alt" content="Content cover image"/>

    <meta name="WT.z_cc_license_type" content=""/>

    <meta name="WT.z_primary_atype" content="Research"/>

    <meta name="WT.z_subject_term" content="Fluorescence imaging;Histology;Neuroscience"/>

    <meta name="WT.z_subject_term_id" content="fluorescence-imaging;histology;neuroscience"/>


    
        <meta property="og:url" content="https://www.nature.com/articles/s41592-020-0823-y"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Nature Methods"/>
        <meta property="og:title" content="Elasticizing tissues for reversible shape transformation and accelerated molecular labeling"/>
        <meta property="og:description" content="The ELAST technology renders large tissue samples elastic through embedding in hydrogel and facilitates sample handling and rapid molecular labeling."/>
        <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig1_HTML.png"/>
    
</head>
<!--[if IE 9]><body class="ie9 article-page"><![endif]-->
<!--[if gt IE 9]><!--><body class="article-page"><!--<![endif]-->

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NWDMT9Q"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>


<div role="banner" class="position-relative cleared z-index-50" data-test="top-containers">
    
    
        <div class="hide-print container background-gray-dark">
            <div class="content mq1200-padded">
                <a href="#content" id="skiplink" class="skiplink contrast-text js-no-scroll" data-track="click"
                   data-track-action="skip to content" data-track-category="header" data-track-label="link">Skip to main
                    content</a>
            </div>
        </div>
    

    
        

        

        
            <div class="container cleared container-type-banner-advert inline-group" data-container-type="banner-advert">
                <div
                    class="hide-print pt10 pb10 mq875-pa0 position-relative z-index-100 clear-float cleared leaderboard-or-billboard-container">
                    <div class="leaderboard-or-billboard-inner ad-with-label">
                        
            
            
                <div id="article-doubleclickad-container">
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-gpt-unitpath="/285/nmeth.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41592-020-0823-y;doi=10.1038/s41592-020-0823-y;subjmeta=1407,1555,1647,2225,245,378,631;kwrd=Fluorescence imaging,Histology,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nmeth.nature.com/article&amp;sz=728x90&amp;c=-1129216185&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41592-020-0823-y%26doi%3D10.1038/s41592-020-0823-y%26subjmeta%3D1407,1555,1647,2225,245,378,631%26kwrd%3DFluorescence imaging,Histology,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nmeth.nature.com/article&amp;sz=728x90&amp;c=-1129216185&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41592-020-0823-y%26doi%3D10.1038/s41592-020-0823-y%26subjmeta%3D1407,1555,1647,2225,245,378,631%26kwrd%3DFluorescence imaging,Histology,Neuroscience"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>
</div>

            
            
        
                    </div>
                </div>
            </div>
        
    

    

    
        <div class="hide-print text-orange content grade-c-show">
            
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
    
        </div>
    


    
        
    <div class="hide-print container background-bluegray contrast-text text14 strong lower mq640-hide position-relative z-index-100"
         data-test="header-breadcrumbs">
        <div class="content cleared mq1200-padded">
            
            <div class="breadcrumbs pin-left">
                <ol class="ma0 cleared clean-list inline-list"><li id="breadcrumb0" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"
                                     itemref="breadcrumb1"><a href="/" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15 "
                                                                                                                 data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="title">nature</span></a></li><li id="breadcrumb1" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"
                                     itemref="breadcrumb2" class="ml6"><a href="/nmeth" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15 "
                                                                                                                 data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature methods"><span itemprop="title">nature methods</span></a></li><li id="breadcrumb2" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"
                                     itemref="breadcrumb3" class="ml6"><a href="/nmeth/articles?type&#x3D;brief-communication" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15 "
                                                                                                                 data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:brief communications"><span itemprop="title">brief communications</span></a></li><li id="breadcrumb3" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a href="/articles/s41592-020-0823-y" itemprop="url" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:article" class="ml6 text-gray"><span itemprop="title">article</span></a></li></ol>
            </div>
            
            
                <p class="pin-right box-sizing header-primary-color nature-research-logo ma0 pa6 hide-text grade-c-hide">A Nature Research Journal</p>
            
        </div>
    </div>


    

    <div class="u-mb-16">
        
    
        <div class="hide-print container js-header-container mb20 position-relative z-index-50" data-ui="header-container"
             data-container-type="header">
            <div class="cleared clear hide-print position-relative background-gradient-brand-primary js-header default-header z-index-100 composite-layer tighten-line-height">
                <div class="banner js-banner content mq1200-padded position-relative">
                    <div class="inner-banner cleared">
                        <div class="main-column small-header-main pin-left">
                            <div class="header-logo-container header-primary-color">
                                <a href="#menu" id="menu-button" class="js-header-menu-button menu-button js-no-scroll"
                                   data-test="menu-button">
                                    <span class="menu-button-icon icon-rotate">
                                        <span class="menu-button-label">Menu</span>
                                    </span>
                                </a>
                                <h1 class="inline-block">
                                    <a href="/nmeth"
                                       class="header-logo inline-block"
                                       data-track="click" data-track-action="home" data-track-category="header" data-track-label="image">
                                        
                                            <img alt="Nature Methods"
                                                 src="//media.springernature.com/full/nature-cms/uploads/product/nmeth/header-14e426c946926ee6925c48c320885794.svg"
                                                 class="grade-c-invisible header-logo-primary js-svg"
                                                 data-png="//media.springernature.com/full/nature-cms/uploads/product/nmeth/header-3d52398c03b7661e5b880eebbd637d40.png"><img alt="Nature Methods"
                                                 src="//media.springernature.com/full/nature-cms/uploads/product/nmeth/header-14e426c946926ee6925c48c320885794.svg"
                                                 class="grade-c-invisible header-logo-secondary js-svg"
                                                 data-png="//media.springernature.com/full/nature-cms/uploads/product/nmeth/header-4772d747277dac66fd8641d6240d8e77.png">
                                        
                                    </a>
                                </h1>
                            </div>
                        </div>
                        <div class="position-absolute position-right small-header-side">
                            <div class="pin-right">
                                
                                    <a href="#search-menu" data-component="tray-button"
                                       class="pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray contrast-text text11 mr1"
                                       data-test="search-link" data-track="click" data-track-category="header"
                                       data-track-action="open tray" data-track-label="button">
                                            <span
                                                    class="icon icon-above icon-search-25x25-white icon-search-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">Search</span>
                                            </span>
                                    </a>
                                    
                                        <a href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;95"
                                           class="mq640-hide pin-left inline-group-top pa10 small-header-icons background-bluegray contrast-text text11 mr1"
                                           data-test="ealert-link" data-track="click" data-track-action="ealert"
                                           data-track-category="header" data-track-label="link">
                                            <span
                                                    class="icon icon-above icon-ealert-25x25-white icon-ealert-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">E-alert</span>
                                            </span>
                                        </a>
                                    
                                    
                                        <a href="https://mts-nmeth.nature.com/cgi-bin/main.plex"
                                           class="mq640-hide inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray contrast-text text11 mr1"
                                           data-test="submit-link" data-track="click" data-track-action="manuscript submission"
                                           data-track-category="header" data-track-label="link">
                                            <span
                                                    class="icon icon-above icon-submit-25x25-white icon-submit-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">Submit</span>
                                            </span>
                                        </a>
                                    
                                    
    <a href="/nams/svc/myaccount"
       id="my-account"
       class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray contrast-text text11"
       data-test="login-link" data-track="click" data-track-action="my account" data-track-category="header" data-track-label="link">
        <span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
            <span class="block mt6 small-header-hide">My Account</span>
        </span>
    </a>
    <a href="https://idp.nature.com/authorize/natureuser?client_id&#x3D;grover&amp;redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41592-020-0823-y"
       id="login-button"
       style="display: none;"
       class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray contrast-text text11"
       data-test="login-link" data-track="click" data-track-action="login" data-track-category="header" data-track-label="link">
        <span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
            <span class="block mt6 small-header-hide">Login</span>
        </span>
    </a>




                                
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    

    </div>

    
    

</div>



<div class="c-banner c-banner--marketing">
    <div class="u-container">
        We’d like to understand how you use our websites in order to improve them. 
        <a class="c-banner__link u-underline"
            href="https://www.surveymonkey.co.uk/r/F2JMQ6L" data-track="click"
            data-track-action="Survey click" data-track-category="Blue Banner"
            data-track-label="10.1038/s41592-020-0823-y">Register your interest.</a>
    </div>
</div>


<div id="content" class="position-relative z-index-1">
    <div class="container cleared container-type-article" data-container-type="article">
        <div class="content position-relative cleared clear mq1200-padded" data-component="article-container">
            <article itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" lang="en">
                <div class="c-article-main-column u-float-left js-main-column" role="main">
                    <div class="c-article-header">
                        <header>
                            <ul class="c-article-identifiers" data-test="article-identifier">
                                
    <li class="c-article-identifiers__item" data-test="article-category">Brief Communication</li>
    
    

                                <li class="c-article-identifiers__item"><a href="#article-info" data-track="click" data-track-action="publication date" data-track-category="article body" data-track-label="link">Published: <time datetime="2020-05-18" itemprop="datePublished">18 May 2020</time></a></li>
                            </ul>

                            
                            <h1 class="c-article-title" data-test="article-title" data-article-title="" itemprop="name headline">Elasticizing tissues for reversible shape transformation and accelerated molecular labeling</h1>
                            <ul class="c-author-list js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-1">Taeyun Ku</a></span><span class="u-js-hide"> 
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0001-9447-7579"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9447-7579</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT)" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Picower Institute for Learning and Memory, MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Picower Institute for Learning and Memory, MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden" data-present-affiliation="true"><meta itemprop="name" content="Graduate School of Medical Science and Engineering, KAIST" /><meta itemprop="address" content="0000 0001 2292 0500, grid.37172.30, Graduate School of Medical Science and Engineering, KAIST, Daejeon, Republic of Korea" /></span></sup><sup class="u-js-hide"> <a href="#nAff10">nAff10</a></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-2">Webster Guan</a></span><sup class="u-js-hide"><a href="#Aff3">3</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Department of Chemical Engineering, MIT, Cambridge, MA, USA" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-3">Nicholas B. Evans</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT)" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Picower Institute for Learning and Memory, MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Picower Institute for Learning and Memory, MIT, Cambridge, MA, USA" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-4">Chang Ho Sohn</a></span><span class="u-js-hide"> 
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0002-7585-1841"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7585-1841</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT)" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Picower Institute for Learning and Memory, MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Picower Institute for Learning and Memory, MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden" data-present-affiliation="true"><meta itemprop="name" content="Center for Nanomedicine, Institute for Basic Science (IBS)" /><meta itemprop="address" content="0000 0004 1784 4496, grid.410720.0, Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden" data-present-affiliation="true"><meta itemprop="name" content="Yonsei University" /><meta itemprop="address" content="0000 0004 0470 5454, grid.15444.30, Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea" /></span></sup><sup class="u-js-hide"> <a href="#nAff11">nAff11</a></sup><sup class="u-js-hide"> <a href="#nAff12">nAff12</a></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-5">Alexandre Albanese</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT)" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-6">Joon-Goon Kim</a></span><span class="u-js-hide"> 
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0002-4620-1989"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4620-1989</a></span><sup class="u-js-hide"><a href="#Aff4">4</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Graduate School of Medical Science and Engineering, Korea Advanced Institute of Science and Technology (KAIST)" /><meta itemprop="address" content="0000 0001 2292 0500, grid.37172.30, Graduate School of Medical Science and Engineering, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-7">Matthew P. Frosch</a></span><sup class="u-js-hide"><a href="#Aff5">5</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="C.S. Kubik Laboratory for Neuropathology, Massachusetts General Hospital and Harvard Medical School" /><meta itemprop="address" content="0000 0004 0386 9924, grid.32224.35, C.S. Kubik Laboratory for Neuropathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA" /></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-8" data-corresp-id="c1">Kwanghun Chung<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-email"></use></svg></a></span><span class="u-js-hide"> 
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0002-8167-3340"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8167-3340</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff3">3</a>,<a href="#Aff6">6</a>,<a href="#Aff7">7</a>,<a href="#Aff8">8</a>,<a href="#Aff9">9</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT)" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Picower Institute for Learning and Memory, MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Picower Institute for Learning and Memory, MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Department of Chemical Engineering, MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="MIT" /><meta itemprop="address" content="0000 0001 2341 2786, grid.116068.8, Department of Brain and Cognitive Sciences, MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Broad Institute of Harvard University and MIT" /><meta itemprop="address" content="grid.66859.34, Broad Institute of Harvard University and MIT, Cambridge, MA, USA" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Center for Nanomedicine, Institute for Basic Science (IBS)" /><meta itemprop="address" content="0000 0004 1784 4496, grid.410720.0, Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea" /></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Yonsei University" /><meta itemprop="address" content="0000 0004 0470 5454, grid.15444.30, Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea" /></span></sup> </li></ul>

                            

                            <p class="c-article-info-details" data-container-section="info">
                                
    <a data-test="journal-link" href="/nmeth"><i data-test="journal-title">Nature Methods</i></a>

                                (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-category="article body" data-track-label="link">Cite this article</a>
                            </p>
                            
    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-inline">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">923 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">63 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41592-020-0823-y/metrics" data-track="click" data-track-action="view metrics" data-track-category="article body" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    



                            
                        </header>

                        
    <div class="js-hide" data-component="article-subject-links">
        
            <h3 class="c-article__sub-heading">Subjects</h3>
            <ul class="c-article-subject-list">
                <li class="c-article-subject-list__subject"><a href="/subjects/fluorescence-imaging" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Fluorescence imaging</a></li><li class="c-article-subject-list__subject"><a href="/subjects/histology" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Histology</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neuroscience" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Neuroscience</a></li>
            </ul>
        
    </div>

                        
    

                        
                    </div>

                    <div data-article-body="true" data-track-component="article body" class="c-article-body">
                        <section aria-labelledby="Abs1" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>We developed entangled link-augmented stretchable tissue-hydrogel (ELAST), a technology that transforms tissues into elastic hydrogels to enhance macromolecular accessibility and mechanical stability simultaneously. ELASTicized tissues are highly stretchable and compressible, which enables reversible shape transformation and faster delivery of probes into intact tissue specimens via mechanical thinning. This universal platform may facilitate rapid and scalable molecular phenotyping of large-scale biological systems, such as human organs.</p></div></div></section>
                        <noscript>
                            
                                
<div class="c-nature-box c-nature-box--side c-nature-box--mobile" data-component="entitlement-box">
    
            
            <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
            
    
        
        <div class="c-pdf-download u-clear-both">
            <a href="/articles/s41592-020-0823-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            </a>
        </div>
        
    

        
    
</div>

                            
                        </noscript>
                        
                            <div class="c-nature-box c-nature-box--side u-display-none" aria-hidden="true" data-component="entitlement-box" id=entitlement-box-mobile>
    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
    
        
        <div class="c-pdf-download u-clear-both">
            <a href="/articles/s41592-020-0823-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            </a>
        </div>
        
    

    
</div>

                        
                        
                            <section aria-labelledby="Sec1"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Engineering tissue properties with the help of hydrogels has facilitated interrogating structural and molecular organization of complex biological systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chung, K. et al. Structural and molecular interrogation of intact biological systems. Nature 497, 332–337 (2013)." href="#ref-CR1" id="ref-link-section-d51879e565">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chen, F., Tillberg, P. W. &amp; Boyden, E. S. Expansion microscopy. Science 347, 543–548 (2015)." href="#ref-CR2" id="ref-link-section-d51879e565_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sylwestrak, E. L., Rajasethupathy, P., Wright, M. A., Jaffe, A. &amp; Deisseroth, K. Multiplexed intact-tissue transcriptional analysis at cellular resolution. Cell 164, 792–804 (2016)." href="#ref-CR3" id="ref-link-section-d51879e565_2">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. Nat. Biotechnol. 34, 973–981 (2016)." href="#ref-CR4" id="ref-link-section-d51879e565_3">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Science 361, eaat5691 (2018)." href="#ref-CR5" id="ref-link-section-d51879e565_4">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ueda, H. R. et al. Tissue clearing and its applications in neuroscience. Nat. Rev. Neurosci. 21, 61–79 (2020)." href="/articles/s41592-020-0823-y#ref-CR6" id="ref-link-section-d51879e568">6</a></sup>. Covalent fusion of tissue and polyacrylamide (pAAm) via a chemical cross-linker allowed preservation of tissue architecture and biomolecules in a cleared tissue-gel<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Chung, K. et al. Structural and molecular interrogation of intact biological systems. Nature 497, 332–337 (2013)." href="/articles/s41592-020-0823-y#ref-CR1" id="ref-link-section-d51879e572">1</a></sup>. This approach, termed CLARITY, has enabled fine-structural and molecular phenotyping of intact biological systems. Further engineering of various tissue-hydrogel properties has expanded the utility of the tissue-gel fusion approach. In particular, the expansion microscopy and magnified analysis of proteome techniques have enabled super-resolution imaging of biomolecules and structures by physically swelling tissue-gel hybrids<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Chen, F., Tillberg, P. W. &amp; Boyden, E. S. Expansion microscopy. Science 347, 543–548 (2015)." href="/articles/s41592-020-0823-y#ref-CR2" id="ref-link-section-d51879e576">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. Nat. Biotechnol. 34, 973–981 (2016)." href="/articles/s41592-020-0823-y#ref-CR4" id="ref-link-section-d51879e579">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Chen, F. et al. Nanoscale imaging of RNA with expansion microscopy. Nat. Methods 13, 679–684 (2016)." href="/articles/s41592-020-0823-y#ref-CR7" id="ref-link-section-d51879e582">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Park, H.-E. et al. Scalable and isotropic expansion of tissues with simply tunable expansion ratio. Adv. Sci. 6, 1901673 (2019)." href="/articles/s41592-020-0823-y#ref-CR8" id="ref-link-section-d51879e585">8</a></sup>.</p><p>However, these approaches suffer from the fundamental limitation of hydrogels: the inverse relationship between structural stability and permeability. To enhance the molecular accessibility of tissue-gel, its permeability needs to be increased, which causes loss of mechanical stability and structural information. Structural stability of tissue-gel can be enhanced by increasing the degree of cross-linking and gel density. Increasing gel density, however, limits the penetration of molecular probes. These innate conflicting properties of hydrogels have restricted the application of the tissue-gel fusion approach to relatively small biological systems, such as zebrafish, rodent brain and small blocks of human tissues. The difficulty associated with using these methods for interrogating large-scale tissues, particularly human brain, prompted us to search for alternative approaches.</p><p>Here we introduce a tissue-hydrogel that offers structural stability while enabling fast transport of molecular probes. We hypothesized that elasticizing tissue would not only render it mechanically durable, but also allow for transient and reversible shape transformation to facilitate molecular access. Elastic hydrogel types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Zhang, Y. S. &amp; Khademhosseini, A. Advances in engineering hydrogels. Science 356, eaaf3627 (2017)." href="/articles/s41592-020-0823-y#ref-CR9" id="ref-link-section-d51879e595">9</a></sup> include hydrogels having a unique chemical or topological cross-linking structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Okumura, Y. &amp; Ito, K. The polyrotaxane gel: a topological gel by figure-of-eight cross-links. Adv. Mater. 13, 485–487 (2001)." href="/articles/s41592-020-0823-y#ref-CR10" id="ref-link-section-d51879e599">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Zhong, M. J., Wang, R., Kawamoto, K., Olsen, B. D. &amp; Johnson, J. A. Quantifying the impact of molecular defects on polymer network elasticity. Science 353, 1264–1268 (2016)." href="/articles/s41592-020-0823-y#ref-CR11" id="ref-link-section-d51879e602">11</a></sup> and double-network hydrogels<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gong, J. P., Katsuyama, Y., Kurokawa, T. &amp; Osada, Y. Double-network hydrogels with extremely high mechanical strength. Adv. Mater. 15, 1155–1158 (2003)." href="#ref-CR12" id="ref-link-section-d51879e606">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sun, J.-Y. et al. Highly stretchable and tough hydrogels. Nature 489, 133–136 (2012)." href="#ref-CR13" id="ref-link-section-d51879e606_1">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Gong, J. P. Materials both tough and soft. Science 344, 161–162 (2014)." href="/articles/s41592-020-0823-y#ref-CR14" id="ref-link-section-d51879e609">14</a></sup>. These gels offer stretchability and toughness, but their syntheses are not suitable for hybridization with biological tissues. For instance, their polymer units are too large to diffuse uniformly into thick tissues before gel formation.</p><p>We discovered that high concentrations (20–60% (wt/vol)) of acrylamide (AAm) alone can polymerize to form an elastic hydrogel in a single synthesis step (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig4">1a</a>). We used orders-of-magnitude-lower concentrations of thermal initiator (1 per 14,000 AAm versus 1 per 73 AAm in CLARITY) and cross-linker (1 per 220,000 AAm versus 1 per 170 AAm reported) to synthesize long polymer chains that naturally undergo entanglement with each other under a high-polymer-density environment. Compared with typical pAAm gels covalently linked by high concentrations of cross-linker (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig1">1a</a>), entangled pAAm gels are formed via physically slippery tangles between growing polymer chains (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig1">1b</a>). Such slip-links endow entangled gels with flexibility and elasticity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Ball, R. C., Doi, M., Edwards, S. F. &amp; Warner, M. Elasticity of entangled networks. Polymer 22, 1010–1018 (1981)." href="/articles/s41592-020-0823-y#ref-CR15" id="ref-link-section-d51879e625">15</a></sup>. Our entangled pAAm gels exhibit stability against physical stresses such as ninefold compression and tenfold stretch (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig4">1b,c</a> and Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM3">1</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: ELAST.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig1_HTML.png?as=webp"></source><img aria-describedby="figure-1-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig1_HTML.png" alt="figure1" loading="lazy" width="685" height="403" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>,<b>b</b>, Schematic drawings describing two hydrogel-forming mechanisms: chemically cross-linked (<b>a</b>) and physically entangled (<b>b</b>) hydrogels. Physical slip links render entangled hydrogel elastic. <b>c</b>, Schematic illustration describing the principle of fast probe delivery in ELASTicized tissue by reversible shape transformation. Biomolecular networks in an ELASTicized tissue follow the transient shape change of the synthetic entangled gel (from left to right) in a reversible way. The timescale (<i>t</i>) for probe delivery into the tissue is proportional to the square of the tissue thickness (<i>d</i>). Transient <i>n</i>-fold thinning of an ELASTicized tissue can accelerate the probe delivery by <i>n</i><sup>2</sup>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>Next, we tested whether in situ synthesis of such elastic pAAm gel can transmute intact tissue into an elastic platform. To prevent any covalent linking between the tissue and pAAm gel, we did not use any chemical fixatives during the gelling process. This strategy allows for the growing pAAm chains to form purely physical entanglements with endogenous biomolecular networks that are constructed by formaldehyde fixation before gelling. We found that the resulting tissue-gel possesses similar mechanical properties as entangled pAAm gels (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig1">1c</a>, top row).</p><p>We applied the technology to a 2-mm-thick, 6 × 8 cm<sup>2</sup> coronal human brain hemisphere slab (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2a,b</a>). Elastic human tissue-gels, as well as mouse tissue-gels, were stretchable and compressible (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2c</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">1</a> and Supplementary Videos <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM4">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM5">3</a>), protecting large intact tissue samples from a variety of mechanical damages. Tissue elasticization was applicable to other matrix-rich organs including heart and colon as well as cerebral organoids (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig5">2</a> and Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM6">4</a>). In addition, we confirmed that biomolecules and tissue architecture are well preserved (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2d,e</a>). We termed this technology ELAST.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Mechanical reinforcement and reversible shape transformation of tissue specimens via ELASTicization.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig2_HTML.png?as=webp"></source><img aria-describedby="figure-2-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig2_HTML.png" alt="figure2" loading="lazy" width="685" height="587" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>,<b>b</b>, A 2-mm-thick coronal human brain hemisphere slab from the occipital pole before (<b>a</b>) and after (<b>b</b>) ELAST transformation and RI-matching. Scale bars, 1 cm. Experiment was repeated two times with similar results. <b>c</b>, Stretchability of an ELASTicized human brain slab. <b>d</b>, Immunolabeling of various proteins as indicated in ELASTicized 200-μm-thick human brain cortical samples. Scale bars, 40 μm. Experiments in <b>c</b> and <b>d</b> were repeated two times with similar results. <b>e</b>, In situ hybridization of <i>GFP</i> and <i>vGluT1</i> mRNAs in an ELASTicized mouse hippocampus expressing EGFP. Scale bar, 40 μm. <b>f</b>, Reversible shape transformation of an ELASTicized 1-mm-thick human brain sample upon twofold stretching in both lateral dimensions. <i>λ</i>, tensile strain. Scale bar, 2 mm. <b>g</b>, Axially projected microscopic images of a prelabeled sample before stretching and after recovery (a representative image from <i>n</i> = 5 samples). Scale bars, 100 μm (insets, 20 μm). <b>h</b>, A mean ± s.d. plot of three-dimensional (3D) geometric deformation measured by the altered distances between feature correspondences in GFAP images at different length scales (<i>n</i> = 5 samples). Dashed line indicates the lateral pixel size of images. r.m.s.e., root mean squared error. <b>i</b>, Reversible shape transformation of a 1.4-mm-thick ELASTicized EGFP-expressing mouse brain sample upon ninefold compression. <i>λ</i>, compression strain. Scale bar, 2 mm. <b>j</b>, Three-dimensionally rendered microscopic images of the sample in <b>i</b> with RI-matching (a representative image from <i>n</i> = 5 samples). Scale bars, 500 μm (insets, 50 μm). <b>k</b>, 3D geometric deformation quantified from EGFP images (<i>n</i> = 5 samples), the same as in <b>h</b>. <b>l</b>, A diagram of the sample-processing procedure for ELASTicization with a suggested timeline. Upper row is for freshly perfused mouse organs, including the brain. Lower row is for about 5-mm-thick specimens from archived human brains that are typically fixed in a formalin solution for a long time. The optimal durations could vary by the degree of fixation and the sample-to-solution volume ratio.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>ELAST allows the entangled biological networks to precisely follow the transient shape transformation of the hydrogel in a fully reversible way. After twofold stretching in both lateral dimensions or tenfold axial compression, ELASTicized samples showed negligible distortion, keeping subcellular architectures intact (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2f–k</a>). These results demonstrate that ELAST enables reversible tissue shape transformation while preserving structural and molecular information of the tissue. The tissue ELASTicization is achieved through several procedural steps including biological networks formation, permeabilization, in situ polymerization of entangled hydrogels and delipidation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2l</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">1</a>). Based on the initial tissue preparation conditions, such as freshly prepared experimental animal tissues or long-term preserved archival human tissue specimens, different processing strategies can be used. The final tissue-gel size is dependent on the refractive index (RI)-matching medium used (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">2</a>), as in other tissue transformation and clearing methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Chung, K. et al. Structural and molecular interrogation of intact biological systems. Nature 497, 332–337 (2013)." href="/articles/s41592-020-0823-y#ref-CR1" id="ref-link-section-d51879e815">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. Nat. Biotechnol. 34, 973–981 (2016)." href="/articles/s41592-020-0823-y#ref-CR4" id="ref-link-section-d51879e818">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hama, H. et al. Scale: a chemical approach for fluorescence imaging and reconstruction of transparent mouse brain. Nat. Neurosci. 14, 1481–1488 (2011)." href="#ref-CR16" id="ref-link-section-d51879e821">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ke, M.-T., Fujimoto, S. &amp; Imai, T. SeeDB: a simple and morphology-preserving optical clearing agent for neuronal circuit reconstruction. Nat. Neurosci. 16, 1154–1161 (2013)." href="#ref-CR17" id="ref-link-section-d51879e821_1">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pan, C. et al. Shrinkage-mediated imaging of entire organs and organisms using uDISCO. Nat. Methods 13, 859–867 (2016)." href="#ref-CR18" id="ref-link-section-d51879e821_2">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Murakami, T. C. et al. A three-dimensional single-cell-resolution whole-brain atlas using CUBIC-X expansion microscopy and tissue clearing. Nat. Neurosci. 21, 625–637 (2018)." href="/articles/s41592-020-0823-y#ref-CR19" id="ref-link-section-d51879e824">19</a></sup>.</p><p>The mechanical properties of ELASTicized tissues open new possibilities in enhancing molecular transport inside the tissue-gel. The tissue thickness, generally the shortest dimension of a tissue, determines the time required for probe penetration (<i>t</i> = <i>d</i><sup>2</sup>/<i>D</i><sub>eff</sub>, where <i>t</i> is dimensionless diffusion timescale, <i>d</i> is tissue thickness and <i>D</i><sub>eff</sub> is the effective diffusivity). Therefore, we reasoned that reducing tissue thickness by transforming its shape would substantially shorten the time required for probe penetration and labeling (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig1">1c</a>).</p><p>To test this, we stretched elastic pAAm gels to thin them by threefold (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3a,b</a>), and incubated them in a solution containing 150-kDa dextran-dye (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3c,d</a>). Fitting the axial dye signal profiles to a one-dimensional (1D) transient diffusion model confirmed that the overall threefold thinning resulted in a ninefold decrease in diffusion timescale as predicted (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3e</a>). We next tested whether thinning of ELASTicized tissues can enhance probe penetration. We stretched 1.5-mm-thick ELASTicized human brain tissues to make them two times thinner and stained them with dye-conjugated antibodies for 30 min. The thinned tissues showed a fourfold increase in antibody penetration depth compared with the unstretched samples (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3f–i</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Thinning-mediated acceleration of probe delivery into ELASTicized tissues.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig3_HTML.png?as=webp"></source><img aria-describedby="figure-3-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig3_HTML.png" alt="figure3" loading="lazy" width="685" height="778" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Control (Ctrl) and stretched (St; twofold in both lateral dimensions) entangled pAAm gels. Colored regions indicate the same areas before stretching. Scale bar, 1 cm. <b>b</b>, Thickness (mean with s.e.m.) change after stretching for 22.5 min (<i>n</i> = 4 gels; two-tailed paired <i>t</i>-test, ***<i>P</i> = 1.94 × 10<sup>−6</sup>, <i>t</i> = 104). <b>c</b>, Delivery of 150-kDa FITC-dextran into control and stretched gels for 45 min. Laterally central regions of top halves of gels were imaged and shown with lateral projections. Stretching was released before taking images. Representative images are shown from four samples in each group. <b>d</b>, Means ± s.d. of depth-wise relative FITC fluorescence (<i>n</i> = 4 gels for each group). Dashed lines show means of modeled fits (<i>R</i><sup>2</sup> (mean ± s.d.), 1.00 ± 0.0011 for Ctrl and 1.00 ± 0.0030 for St). <b>e</b>, A comparison of diffusion timescales (means with s.e.m.) drawn from the fitted models in <b>d</b> (two-tailed unpaired <i>t</i>-test, ***<i>P</i> = 3.75 × 10<sup>−5</sup>, <i>t</i> = 26). <b>f</b>, Schematic drawings of antibody delivery into tissue samples in different thinning approaches. Representative photos (10, 5, 5 and 5 tissue samples in the order shown) of ELASTicized 1-mm-thick human brain cortical samples in the conditions are shown together. Yellow and blue dashed lines indicate tissue and hydrogel areas, respectively. A 40% 1,2-dimethoxyethane solution was used for contraction (Co). Scale bar, 2 mm. <b>g</b>, A comparison of thicknesses (<i>n</i> = 10, 5, 5 and 5 samples; mean with s.e.m.; one-way analysis of variance, ***<i>P</i> = 4.95 × 10<sup>−24</sup>, <i>F</i> = 1,310; post hoc Tukey’s test yielded <i>P</i> &lt; 0.05 for a comparison between St and Co, and <i>P</i> &lt; 0.001 for other pairs). <b>h</b>, Laterally projected images of central tissue regions showing immunolabeling depths after 30 min of incubation with a fluorophore-conjugated anti-GFAP antibody. A reference image was obtained by exposing a sample’s cut surface to the antibody solution for 1 d and imaging the cut surface vertically. Experiments were repeated two times with similar results. <b>i</b>, A comparison of depth-wise signal intensities among images in <b>h</b>. <b>j</b>, A schematic drawing of the cyclic compression device. UHMW PE, ultrahigh molecular weight polyethylene. <b>k</b>, An example of a cyclic compression protocol. Each cycle consists of a compression period for acceleration and a release period for flushing with a solution. <b>l</b>, Three-dimensionally rendered images of an ELASTicized 3-mm-thick mouse sample expressing EGFP before and after about 1,500 times of cyclic sixfold compression over 1 d (a representative image from <i>n</i> = 4 samples). <b>m</b>, A mean ± s.d. plot of 3D geometric deformation quantified from EGFP images (<i>n</i> = 4 samples). <b>n</b>, A three-dimensionally rendered image and selected planes, showing rapid full-depth immunolabeling of an originally 5-mm-thick human brain cortical sample using cyclic compression for 1 d. The sample was compressed to 16% thickness for 40 s and released for 10 s for each cycle. A central region of the sample was imaged after RI-matching. Selected plane images were individually contrast-adjusted. Scale bar, 500 μm (selected planes, 200 μm). <b>o</b>, Immunolabeling of neuronal cell types and projections in a human cerebral cortex. NF-L, neurofilament light chain. Scale bar, 100 μm. Experiments in <b>n</b> and <b>o</b> were not repeated. <b>p</b>, A typical procedure and timeline for immunolabeling of an ELASTicized 5-mm-thick human brain sample assisted by thinning. Ab, antibody.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="/articles/s41592-020-0823-y/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>Additional thinning of ELASTicized tissue could further enhance probe delivery and shorten the labeling time. We discovered that 40% 1,2-dimethoxyethane in water can isotropically contract ELASTicized tissues by 1.7-fold in each dimension (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3f,g</a>). When applied to stretched ELASTicized tissues, this solution elicited a 1D contraction that led to additional 3.4-fold thinning, making the samples thinned by 6.3-fold overall. This synergistic two-step thinning enabled complete immunostaining of the 1.5-mm-thick ELASTicized human tissue within only 30 min (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3h,i</a>). However, when applied to unstretched tissues, isotropic contraction by 1,2-dimethoxyethane suppressed antibody penetration likely due to reduced pore size.</p><p>Although stretching combined with chemical-mediated shrinkage can shape-transform ELASTicized tissues to accelerate probe delivery, implementing this method is not straightforward. Stretching requires extra gel surrounding the tissue to interface with a stretching apparatus. In addition, 1,2-dimethoxyethane could disrupt probe–target binding. These complications together could limit the utility of this approach. We therefore sought an alternative technique for sample-thinning. We directly compressed an ELASTicized sample, instead of stretching (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">3</a>), and periodically released the sample for flushing to avoid depletion of probes. We built a motorized device to automate this cycling (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3j,k</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">4</a> and Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM7">5</a>). Fivefold cyclic compression caused no noticeable tissue damage over 1 d (~1,500 cycles) and no substantial change in the quality of immunostaining (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3l,m</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">5</a>). The increase in antibody penetration depth in cyclic compression was slightly less than that in stretching-based thinning probably due to the time required for the decompression steps (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">6</a>).</p><p>We next applied the compression method for immunolabeling of a 5-mm-thick human brain sample within only 1 d (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3n</a> and Supplementary Video <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM6">6</a>), which is over 100 times faster than what we previously reported (1 d for labeling 0.5-mm-thick human brain block)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Chung, K. et al. Structural and molecular interrogation of intact biological systems. Nature 497, 332–337 (2013)." href="/articles/s41592-020-0823-y#ref-CR1" id="ref-link-section-d51879e1044">1</a></sup>. In addition, we demonstrated the potential of this approach as a scalable platform for mapping neuronal connectivity in higher animal models and human samples by visualizing neuronal cell types and their projections in human brain tissues as well as in cerebral organoids (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig3">3o,p</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">7</a>, Supplementary Videos <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM9">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM10">8</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">1</a>).</p><p>In summary, we report a tissue transformation technique that simultaneously enhances mechanical properties and chemical accessibility of tissue by elasticizing it. We discovered that elastic homo-pAAm gel can be synthesized by promoting physical entanglement of long and densely packed pAAm chains. In situ synthesis of such an elastic gel transforms intact tissue into a deformable matrix that can be stretched or compressed without mechanical damage.</p><p>Transient and reversible thinning of ELASTicized tissue by cyclic compression enables fast molecular labeling. This mechanical approach has the potential to accelerate transport of a range of molecules without affecting their chemical and functional properties. As a step toward such extension, we demonstrated the compatibility of ELAST with a variety of antibodies, nuclear and lipophilic dyes, and messenger RNA probes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig2">2d,e</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">8</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">3</a>).</p><p>The current ELASTicization protocol takes about 20 d to process fresh mouse organs or archived human tissues with similar tissue thickness, but once ELASTicization is completed, half-centimeter-thick tissues can be stained within 1 d. Such rapid staining could be particularly useful for biological applications requiring multiple rounds of staining and imaging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Murray, E. et al. Simple, scalable proteomic imaging for high-dimensional profiling of intact systems. Cell 163, 1500–1514 (2015)." href="/articles/s41592-020-0823-y#ref-CR20" id="ref-link-section-d51879e1085">20</a></sup>. In addition, we found that the mechanical stability of ELAST enables cryo-sectioning of intact ELASTicized tissues into 3–10-μm-thick sections after immunofluorescence observation, allowing downstream histology such as hematoxylin and eosin staining (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41592-020-0823-y#Fig6">3</a>).</p><p>We envision that ELAST will expedite scaling of tissue phenotyping approaches to larger systems by simultaneously overcoming two major bottlenecks: slow molecular labeling and low tissue integrity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Zhao, S. et al. Cellular and molecular probing of intact human organs. Cell 180, 796–812 (2020)." href="/articles/s41592-020-0823-y#ref-CR21" id="ref-link-section-d51879e1095">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Kubota, S. I. et al. Whole-body profiling of cancer metastasis with single-cell resolution. Cell Rep. 20, 236–250 (2017)." href="/articles/s41592-020-0823-y#ref-CR22" id="ref-link-section-d51879e1098">22</a></sup>. For instance, the lateral dimension of human organ slices is orders of magnitude larger than that of rodent organ sections, rendering human specimens more prone to mechanical damage than animal tissues with the same thickness. ELAST not only transforms tissue into a virtually indestructible platform, it also enables labeling thicker tissues at a faster rate. Together with the versatility and simplicity of the method, ELAST will facilitate investigation of higher animal models and clinical human samples.</p></div></div></section><section aria-labelledby="Sec2"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Methods</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Reagents and reagent preparation</h3><p>Solutions for making pAAm hydrogels were prepared using AAm (A9099; MilliporeSigma), 2% <i>N</i>,<i>N</i>′-methylenebisacrylamide (MBAA) solution (161-0142; Bio-Rad Laboratories) and VA-044 (Wako Chemicals). A 10% (wt/vol) VA-044 stock solution was prepared in deionized (DI) water, aliquoted and kept at −20 °C. Frozen VA-044 stock aliquots were thawed and used only during the same day. For hydrogel experiments, the gel monomers and the VA-044 stock solution were directly dissolved in DI water as needed. For ELAST tissue preparation, an ELAST stock solution without VA-044 was prepared in DI water (0.1% volume was left to add the VA-044 stock solution later) with 30% (wt/vol) AAm, 0.0003% MBAA (using 2% solution), 0.2% (wt/vol) Triton X-100 (0694; VWR Scientific) using a 10% (wt/vol) stock solution in DI water and 1× PBS using Gibco 10× PBS (70011-044; Life Technologies). The ELAST stock solution was kept at 4 °C for up to 6 months and added by 0.1% volume of the VA-044 stock solution for each use to make a final ELAST solution with 0.01% (wt/vol) VA-044. Perfusion and Off solutions used for the stabilization underharsh conditions via intramolecular epoxide linkages to prevent degradataion (SHIELD) were prepared using polyglycerol 3-polyglycidyl ether (GE38) provided by CVC Thermoset Specialties of Emerald Performance Materials<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Park, Y.-G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. Nat. Biotechnol. 37, 73–83 (2019)." href="/articles/s41592-020-0823-y#ref-CR23" id="ref-link-section-d51879e1120">23</a></sup>. SHIELD perfusion solution was prepared with 2.5% (wt/vol) GE38 and 4% paraformaldehyde (PFA) using a 32% solution (15714-S; Electron Microscopy Sciences) in 0.1 M sodium phosphate (diluting 1 M sodium phosphate buffer (P2072; Teknova)) and 50 mM sodium chloride (diluting 5 M sodium chloride (S5845; Teknova)). A supernatant was collected and used by vortexing the solution for 1 min and repeating (1) centrifugation for 10 min at 4 °C and 7,200 relative centrifugal force (r.c.f.) and (2) collecting a supernatant until the supernatant was clear. SHIELD-Off solution was prepared by omitting PFA from the SHIELD perfusion solution. SHIELD-On solution was prepared by combining 2 ml of 1 M sodium carbonate (S7795; MilliporeSigma) with 0.02% (wt/vol) sodium azide, 2 ml of 1 M sodium bicarbonate (S5761; MilliporeSigma) with 0.02% (wt/vol) sodium azide and 36 ml of DI water. A clearing solution was prepared by dissolving 6% (wt/vol) SDS (75746; MilliporeSigma), 50 mM sodium sulfite (S0505; MilliporeSigma) and 0.02% (wt/vol) sodium azide (S2002; MilliporeSigma) in 0.1 M phosphate buffer, and the pH was adjusted to 7.4. Washing solutions were prepared as 0.1% (wt/vol) Triton X-100 in PBS (diluting 10× PBS (P0496; Teknova)) either with (PBST) or without (PBST without azide) 0.02% (wt/vol) sodium azide. PBS with 0.02% (wt/vol) sodium azide (PBS with azide) was also used as a washing solution. A blocking solution was prepared by adding 5% volume of normal goat serum (ab7481; Abcam) to PBST. See Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">3</a> for antibody and probe purchase and usage for mouse and human brain tissues. A 1,2-dimethoxyethane solution was prepared by mixing 1,2-dimethoxyethane (307432; MilliporeSigma) and PBST in a volume-to-volume ratio of 4:6. Antibody fixation solution was prepared in PBS with 4% PFA using a 32% solution and Gibco 10× PBS. Two types of RI-matching immersion media were used: (1) an immersion medium of a lower contraction power, named exPROTOS, and (2) an immersion medium of a higher contraction power, named dPROTOS. exPROTOS was prepared by dissolving 125 g iohexol, 3 g diatrizoic acid and 5 g <i>N</i>-methyl-<span class="u-small-caps">d</span>-glucamine in 100 ml of DI water, followed by RI adjustment to 1.458 by adding DI water, and then mixing 90% volume of the medium with 10% volume of 2,2′-thiodiethanol (16782; MilliporeSigma), which yielded a final RI of 1.465. dPROTOS (RI = 1.52) was prepared by mixing 150 ml of 2,2′-thioodiethanol, 150 ml of dimethyl sulfoxide and 200 ml of DI water, and dissolving 530 g of iohexol in the mixture. AAm, MBAA, 1,2-dimethoxyethane and PFA are toxic chemicals requiring caution and should be handled in a chemical hood.</p><h3 class="c-article__sub-heading" id="Sec4">Entangled hydrogel experiments</h3><p>All hydrogels were polymerized in 50-ml conical tubes (referred to as polymerization tubes). The common preparation sequence is as follows: (1) a polymerization tube was purged with a nitrogen gas (NI UHP300; Airgas East) at 80 kPa (at a regulator level) for 1 min; (2) a gel solution was put in the tube; (3) the tube was additionally purged with a nitrogen gas for 15 s and sealed using a screw cap connected to 12-psi nitrogen gas; and (4) the tube was vertically placed in a custom-built heating device. Gel solutions to confirm dense gel formation without MBAA were prepared in a fixed ratio between monomer and initiator concentrations. Gel solutions of 2 ml were prepared in 15-ml conical tubes and bubbled using a nitrogen gas-connected glass capillary (1B100F-4; World Precision Instruments) at a low gas pressure for 1 min. The solutions (1 ml) were transferred to polymerization tubes. Low-concentration (4% (wt/vol) AAm with and without 0.05% (wt/vol) MBAA) solutions with 0.04% (wt/vol) VA-044 were polymerized at room temperature (RT), and high-concentration (40% (wt/vol) AAm with and without 0.5% (wt/vol) MBAA) solutions with 0.4% (wt/vol) VA-044 were polymerized on ice to avoid runaway heat generation. The tubes were left for 1 d and photos were taken. To test compression, the top of each 15-ml conical tube was cut into a 2-cm-long cylinder, and the original top was superglued to a squared no. 1 coverslip (48393070; VWR Scientific). Gel solutions (40% (wt/vol) AAm and 0.005% (wt/vol) VA-044 with and without 0.5% (wt/vol) MBAA) of 2 ml were prepared in 15-ml conical tubes and nitrogen gas-bubbled for 1 min. The solutions (1.5 ml) were transferred to the cut cylinders. The cylinders were put in polymerization tubes. Polymerization was completed at 37 °C after about 15 min, and the tubes were kept overnight for stabilization. Gels of 9-mm thickness were collected, and each was placed on a vise jaw. Two fragments of 1-mm-thick slide glasses were left on the jaw as a thickness guide. The vice was tightened and then released as a compression demonstration. To test stretching, 2-ml gel solutions (5 M AAm and 0.005% (wt/vol) MBAA) were prepared, transferred to 1-ml syringes (SS-01T; Fisher Scientific) and nitrogen gas-bubbled for 1 min. Each syringe was sealed with Parafilm and covered with a Blu-Tack adhesive (Bostik) ball. The syringes were put in polymerization tubes filled with DI water and polymerized and stabilized at 35 °C for 1 d. The syringes were cut, and the gels were taken out and used for a stretching demonstration.</p><h3 class="c-article__sub-heading" id="Sec5">Diffusion experiment and modeling using hydrogels</h3><p>Two-millimeter-thick cartridges were made of two 1-mm-thick slide glasses spaced by supergluing four stacks of two slide glass fragments near the four edges. The lateral and bottom edges were taped, and further secured with superglue. A gel solution of 30% (wt/vol) AAm, 0.0003% (wt/vol) MBAA and 0.01% (wt/vol) VA-044 in DI water was poured into the cartridges and polymerized in polymerization tubes at 33 °C for 6 h. After swelling extracted gels in excess DI water at 37 °C for 3 d with daily exchange for fresh water, swollen 4-mm-thick gel slabs were cut into squares (2 × 2 cm<sup>2</sup>). Any top regions having less-swollen sizes were excluded. A stretching apparatus was made with four 56-mm-long, 1.6-mm-diameter stainless steel rods (cut from 1263K24; McMaster-Carr) and four 38-mm-long tube pieces cut from polypropylene 1,250-μl pipette tips having two holes in a 32-mm-long interval. Each of the squared gels was first pierced using 25-gauge syringe-needles (305125; BD) along the four lateral edges in 16-mm-long intervals, then the rods were inserted into these piercings and finally the gels were stretched with the aid of the pipette tip pieces. A micrometer (395-371-30; Mitutoyo America) was used to measure the thicknesses of unstretched gels and stretched gels after immersing in PBST for 22.5 min. Each thickness was measured three times, and the median was reported.</p><p>A 150-kDa fluorescein isothiocyanate (FITC)-dextran (FD150S; MilliporeSigma) stock solution was prepared at 10 mg ml<sup>−1</sup> in DI water, aliquoted and frozen at −20 °C. An FITC-dextran solution in PBST was freshly prepared at 0.2 mg ml<sup>−1</sup> using the stock solution. Control and stretched gels were incubated in 40-ml FITC-dextran solutions on a gentle shaker for 45 min at RT. The stretched gels were recovered by removing the apparatus immediately after the incubation. Each of the stained gels was mounted in an imaging chamber made of a slide glass, a WillCo dish (HBSB-5030; WillCo Wells) and two spacers using four slide glasses. The topmost 3-mm thickness of a central region (0.3 × 0.3 mm<sup>2</sup>) of the mounted gel was imaged using the Olympus FV1200MPE laser-scanning confocal microscope with a 10×, 0.3-numerical aperture (NA) water-immersion objective (UMPLFLN 10XW) and a 488-nm argon laser. Mounting and imaging were completed within 3 min. The fluorescence signals were subtracted by depth-wise background signals obtained from an unstained gel imaged in the same way.</p><p>A 1D transient diffusion model was used to fit the FITC-dextran distribution profiles:</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{\partial }{{\partial t}}C\left( {x,t} \right) = D_{\mathrm{eff}}\frac{{\partial ^2}}{{\partial x^2}}C\left( {x,t} \right),$$</span></div></div><p>where <i>C</i>(<i>x</i>, <i>t</i>) is the nondimensional relative FITC-dextran concentration at depth <i>x</i> at time <i>t</i> and <i>D</i><sub>eff</sub> is the effective diffusivity. Because of symmetry, the lower half of the gel was assumed to have a mirror distribution of the upper half. The initial and boundary conditions for the partial differential equation are:</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$C\left( {0,t} \right) = 1,$$</span></div></div><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{\partial }{{\partial x}}C\left( {T,t} \right) = 0,$$</span></div></div><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$C\left( {0 &lt; x \le T,0} \right) = 0,$$</span></div></div><p>where <i>T</i> is the half-thickness of a gel. The equation was numerically solved using custom MATLAB (R2016a; MathWorks) scripts to derive <i>D</i><sub>eff</sub> using least-squares curve fitting. The surface data points showing reduced concentration possibly due to regurgitation were excluded from the fitting and then reconstructed from the fitted solution for display. Obtained <i>D</i><sub>eff</sub> values were converted to relative dimensionless diffusion timescale for a comparison.</p><h3 class="c-article__sub-heading" id="Sec6">Preparation of human brain samples</h3><p>Six coronal human brain hemisphere slabs (about 1-cm thickness) in a formalin solution were obtained from three autopsy brains at the Neuropathology Core of the Massachusetts Alzheimer Research Center. The brains were collected and banked in accordance with approval from the local Institutional Review Board. Subjects with no evidence of neurological disease on clinical grounds at the time of death and no evidence of notable disease processes upon full neuropathologic examination were selected. The formalin solution was replaced with PBS with azide upon arrival, and the slabs were stored at 4 °C until use. Whole-area coronal hemisphere slabs of 2-mm thickness were obtained by microtome-sectioning of the original thick slabs using a custom-built flexure-based vibrating-blade microtome. Each of the thick slabs was embedded in a 1.5% (wt/vol) agarose gel in PBS, where any gel on the bottom surface was removed, and superglued on a glass plate. The glass plate was superglued on a microtome chamber, and the chamber was filled with PBS. A frequency of 60 Hz and a blade propagation speed of 0.3 mm s<sup>−1</sup> were used. For large tissue blocks used for compression testing and thick-tissue labeling, cortical regions were manually cut using a razor blade to include a thickness of up to 5 mm from the pial surface. For measurement of tissue size changes, 2–3-mm-thick cortical blocks were obtained by manual cutting. Horizontal tissue samples of 1-mm thickness used for antibody delivery comparison and stretching-related deformation analysis were obtained by slicing with a vibrating-blade microtome (VT1200; Leica Biosystems) of several-millimeters-thick horizontal cortical blocks that were manually cut. The pial surface region (200–300-μm thickness) of each sample was trimmed off. The obtained 1-mm-thick samples were manually cut into square pieces (2 × 2 mm<sup>2</sup>). A coronal slice having an original thickness of about 1.5 mm was used for the antibody compatibility test.</p><h3 class="c-article__sub-heading" id="Sec7">ELASTicization of archived human brain specimens</h3><p>The long-term formalin-fixed human tissue samples were first permeabilized using clearing solution, for which a 70 °C overnight step was followed by a 56 °C step, if any. See Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">1</a> for details of each experiment. Permeabilized samples were washed with PBST without azide. At this point, it is important to ensure the removal of most of the azide and sulfite from the tissues, because these chemicals quench radicals during AAm polymerization and impede gel formation. Samples were then incubated in an ELAST solution at 4 °C overnight (1-mm-thick samples), for 5 d (5-mm-thick samples) or for 8–9 d (2-mm-thick slabs). Other than for large-area slabs, polymerization cartridges were prepared using two 1-mm-thick slide glasses spaced at four edges by superglued stacks of slide glass and coverslip fragments that matched the thickness of each sample in the ELAST solution. After taping with transparent plastic tape along lateral and bottom edges and then tailoring and supergluing the tape, samples were inserted into the cartridges. The space in each cartridge was filled carefully with the same ELAST solution used to incubate the tissues, ensuring that there were no apparent bubbles. The cartridges were put in 50-ml conical tubes that were prepurged with a nitrogen gas at 12 psi for 1 min. The tubes were then additionally purged for 15 s and sealed with screw caps connected to a nitrogen gas at 12 psi. Samples were polymerized at 20–37 °C for 6 h. In a cold environment, each 2-mm-thick hemisphere slab incubated in ELAST solution was placed on a glass plate (260232; Ted Pella) having superglued 2-mm-thick spacers (stacks of two slide glass fragments) at four corners of the plate. Another glass plate was first placed on top of the sample at a height of 3 mm, guided by two stacks of three slide glasses. Then, the space above the sample was filled with the same solution, and the cover plate was lowered to 2-mm height. The glass plates were taped, leaving a small opening at one corner, and sealed using a superglue, which was cured at 4 °C. The cartridge space was filled with the same solution using a 20-ml needle-syringe while avoiding bubble-trapping. The cartridge was placed in a plastic bag with a zipper lock. The bag was purged with nitrogen gas at 12 psi for 10 min, closed, placed in a water bath where the temperature was maintained at 33 °C, and polymerized and stabilized overnight. Embedded samples were cut from the gels using a razor blade to have adequate gel margins and cleared in clearing solution at 37 °C overnight (1-mm-thick samples) or at 56 °C for 5 d (5-mm-thick samples) or 2 d (2-mm-thick slabs, in a 500-ml glass bottle) with exchange of the solution every 1 or 2 d. Fully cleared samples were washed with PBST at 37 °C for 6 h (1-mm-thick samples) or 2–3 d (other samples) with exchange of the solution two or three times. Gels containing samples for the antibody delivery speed comparison were cut into squares (6 × 6 mm<sup>2</sup>) using a razor blade so as to have one tissue sample for each square at the center. Thick samples used for compression testing and thick-tissue labeling were carefully isolated from the surrounding gels.</p><h3 class="c-article__sub-heading" id="Sec8">Preparation and ELASTicization of mouse tissue samples</h3><p>All experimental protocols were approved by the Massachusetts Institute of Technology (MIT) Institutional Animal Care and Use Committee and the Division of Comparative Medicine and were in accordance with guidelines from the National Institutes of Health. All experiments using mice were conducted in strict adherence to the ethical regulations of MIT Institutional Animal Care and Use Committee and the Division of Comparative Medicine. Thy1-GFP-M transgenic mice (2–4 months old, male and female) were housed in a 12-h light/dark cycle with unrestricted access to food and water. Mice were transcardially perfused with 20 ml of ice-cold PBS followed by 20 ml of ice-cold SHIELD perfusion solution at a flow rate of 5 ml min<sup>−1</sup>. Organs were extracted and incubated in 15–20 ml of the same perfusion solution at 4 °C for 48 h with shaking. The organs were transferred to 20 ml of prechilled SHIELD-Off solution and incubated at 4 °C for 24 h with shaking. The organs were placed in 40 ml of SHIELD-On solution and incubated at 37 °C for 24 h. SHIELD-fixed organs were washed with PBS with azide at RT overnight. Brains were split into hemispheres using a razor blade. The brain hemispheres and other organs were delipidated passively in clearing solution at 37 °C for 12–14 d, 45 °C for 10 d, or 45 °C for 2 d and 56 °C for 3 d until the opaque core disappeared. Cleared tissues were washed with 50 ml of PBST without azide or with 0.02% (wt/vol) thimerosal (for in situ hybridization; T5125; MilliporeSigma) at 37 °C for 2–3 d with three to four solution exchanges. The tissues were incubated in ELAST solution at 4 °C for 3–5 d with daily solution exchange. One or two brain hemispheres or other organs were inserted into a polymerization cartridge, and polymerization was performed at 20–37 °C for 6 h. Embedded samples were cleared in clearing solution at 37 °C for 5–9 d or 56 °C for 4 d with solution exchange every 2 d, and then washed with PBST at 37 °C for 2–4 d with daily solution exchange. Surrounding gels were carefully removed from the samples during or after washing. The 1-mm-thick mouse coronal brain hemisphere slices used for demonstrations of stretching, measurement of size changes and antibody screening were prepared by embedding microtome-sectioned 1-mm-thick slices. See Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM1">1</a> for the detailed protocol for each of the mouse samples in the current study.</p><h3 class="c-article__sub-heading" id="Sec9">Thinning of ELASTicized tissue samples</h3><p>A 4.5-mm-thick ELASTicized human sample used for the compression demonstration was placed on a 15 cm (width) × 15 cm (height) × 1.3 cm (thickness) glass plate (CG-1904-18; Chemglass Life Sciences) together with two spacers of approximately 0.42-mm thickness using stacks of three no. 1 coverslips at two lateral edges of the plate. The sample was covered with another plate and photos were taken before and after compression. The thickness of the coverslip stacks was the target final compressive thickness of the sample. Mouse organs and cerebral organoids used for the compression demonstration were tested similarly, but spacers were not used. ELASTicized mouse brain samples for the quantification of compression-related deformation were obtained from an ELASTicized mouse brain hemisphere: one hemisphere was manually cut using a razor blade to obtain a 3-mm-thick block, and this block was microtome-sliced to get a 1.4-mm-thick slice; the slice was chopped into several pieces using a razor blade. Obtained pieces were compressed between two glass plates with dimensions of 75 mm (width) × 51 mm (height) × 1.2 mm (thickness) (71862-01, Electron Microscopy Sciences). A no. 1 coverslip was placed on each of the lateral edges of the bottom glass plate to guide a tenfold compression. Samples were compressed ten times for at least 1 min, and photos were taken before and after each compression. The mouse brain samples to measure the compressed area were obtained by chopping an ELASTicized 1-mm-thick brain slice. The samples having a thickness greater than 2 mm were first compressed to 2 mm between two 1.2-mm-thick glass plates with two stacks of two slide glasses as spacers to ensure all top and bottom surface regions were in contact with the plates. After taking photos, the samples were further compressed to 1 mm by reducing the heights of the spacers. After taking photos again, compressed areas were measured using ImageJ (National Institutes of Health) and compared between 2-mm and 1-mm compressions.</p><p>A stretching apparatus was made with four 14.5-mm-long needles cut from 25-gauge syringe-needles and 10.5-mm-long tube pieces cut from polypropylene 10-μl pipette tips having two holes in an 8-mm-long interval. ELASTicized 1-mm-thick human samples were pierced by the needles along four lateral edges in an 8-mm-long interval and stretched with the aid of the pipette tip pieces. Samples for the quantification of stretching-related deformation were stretched overnight at 37 °C. Photos were taken before stretching, after overnight stretching and after &gt;2-h recovery. The thicknesses of ELASTicized 1-mm-thick human samples were measured by a micrometer three times, and the median was chosen for each sample. The thicknesses of stretched samples were measured after stretching followed by incubation in 2 ml of PBST at RT for 3 h. The thicknesses of contracted samples were measured after incubation in a 10-ml 1,2-dimethoxyethane solution at 37 °C overnight and then at RT for 6 h for a full incubation to dismiss a transient opaqueness. After incubating stretched samples in a 5-ml 1,2-dimethoxyethane solution at RT for 1 h, exchanging the solution, and incubating at 37 °C overnight and then at RT for 1 h, the thicknesses of the stretched and contracted samples were measured.</p><h3 class="c-article__sub-heading" id="Sec10">Labeling of thin ELASTicized brain samples</h3><p>For the antibody compatibility test with human brain tissue, an approximately 1.5-mm-thick human brain coronal slice was ELASTicized and then microtome-sliced to obtain 200-μm-thick sections. A slow blade propagation speed (0.1 mm s<sup>−1</sup>) and a high amplitude (1 mm) were used to slice the elastic sample. The sections were cut into small pieces (about 2 × 2 mm<sup>2</sup>) and put in a 24-well plate. Samples were incubated in PBST with primary antibodies overnight, washed with PBST three times for 2 h each, incubated in PBST with secondary antibodies overnight and washed with PBST twice for 2 h each. The well plate was placed on a gentle shaker at 37 °C during the procedure.</p><p>For the antibody compatibility test with mouse brain tissue, 1-mm-thick SHIELD-protected mouse brain hemisphere slices were used. Control samples were first vibratome-sliced into 100-μm-thick sections at this step. Both 100-μm-thick control samples and 1-mm-thick samples to be ELASTicized were delipidated in clearing solution (200 mM SDS in pH 9 borate buffer) for the same amount of time (1.5 d at 45 °C). One-millimeter-thick samples were ELASTicized, vibratome-sliced into originally 100-μm-thick sections, additionally delipidated for 8 h at 37 °C, washed at 37 °C overnight and incubated in blocking solution at 37 °C for 3 h. Small pieces were obtained from both groups of samples, and each pair was immunostained separately at 37 °C: primary antibody (1:300) overnight, washing with PBST three times for 8 h, secondary antibody (1:300) overnight and washing with PBST three times for 6 h. Lectin (1:300) was added at the primary antibody staining step. DAPI (D1306; Life Technologies) was dissolved in DI water at a concentration of 5 mg ml<sup>−1</sup>, and this stock solution was used at a 1:10,000 dilution during the second time of washing after secondary antibody staining. Dil (D282; Life Technologies) was prepared at a concentration of 2.5 mg ml<sup>−1</sup>, and the solution was used at a 1:1,000 dilution at the secondary antibody staining step.</p><p>For the antibody delivery speed comparison, ELASTicized 1-mm-thick human samples were stained with a 10-μl anti-glial fibrillary acidic protein (GFAP) antibody conjugated with Alexa Fluor 594 in a 50-ml conical tube with 1 ml of PBST on a gentle shaker at RT for 30 min. Stretched samples were put in the antibody solution immediately after stretching. Stretched and contracted samples were prepared by incubating stretched samples in a 3-ml 1,2-dimethoxyethane solution at RT for 30 min (exchanging the solution once after 10 min). After exchanging again with fresh 1-ml 1,2-dimethoxyethane solution, antibody was added. Contracted samples were prepared by incubating samples in a 3-ml 1,2-dimethoxyethane solution at 37 °C overnight (exchanging once) and then at RT for 3 h. Antibody-stained samples were washed with PBST at RT for 30 min. All stretched samples were recovered from stretching and left in PBST for 30 min before imaging.</p><p>To compare cyclic compression with other methods in regard to antibody delivery speed in the same setting, a manual cyclic compression setting was used. A plastic film bag of about 6 cm (height) by 2 cm (width) was made using an impulse heat sealer. After putting a sample into the bottom of the bag, a horizontal midline was sealed leaving a small hole at one side. PBST (1 ml) was added to the bottom chamber, and 10 μl of anti-GFAP antibody was added to the top chamber. The bag was then completely sealed, and the lower chamber area was placed between two slide glasses with spacers at two sides that matched a whole thickness of the bag with a sample compressed by sixfold. A manual cyclic compression was applied for 30 min by repeating a compression for 50 s that pushed the solution out to the upper chamber and a release for 10 s by compressing the upper chamber. After staining, the bag was cut with a razor blade, and the sample was taken out and washed with PBST for 2 h before imaging.</p><p>One ELASTicized 1-mm-thick sample was used to test the compatibility of thinning methods with cell-type and projection markers. The sample was stretched twofold laterally and then antibody-stained. A 1,2-dimethoxyethane solution (35:65 volume ratio) was used for anti-neurofilament light chain antibody, and other primary and secondary antibodies were stained in PBST.</p><h3 class="c-article__sub-heading" id="Sec11">Thick-tissue immunolabeling with cyclic compression</h3><p>An ELASTicized 5-mm-thick human sample (lateral dimensions of 12 and 7 mm) was photobleached to remove autofluorescence from lipofuscin pigments. A photobleaching device was built by wrapping a 1-l glass beaker with a strip of white light-emitting diodes (Ledmo SMD5630; Shenzhen Xiangliang Lighting). The sample in PBST was placed in the center of the horizontally positioned device at 4 °C for 3 d, then preblocked in a blocking solution at 37 °C for 1 d on a gentle shaker and then kept in the same solution until antibody-staining. A cyclic compression device was custom-built using a Nema 23 bipolar stepper motor (23HS16-0084S; OSM Technology) connected to a Raspberry Pi 3 B+ motherboard (Raspberry Pi Foundation) with a dedicated stepper motor controller (2348; Adafruit). The axial movement part of the device consisted of a 60-mm cage system (two LCP02 and one LCP03; Thorlabs) and custom-made polycarbonate plates. One plate was fixed in place while the other was driven with a leadscrew cut from an M6-threaded rod (1078N11; McMaster-Carr), which was coupled to the shaft of the stepper motor. A sample container bag with a width of 5.5 cm and height of 15 cm was made of an ultrahigh-molecular-weight polyethylene film (85655K12; McMaster-Carr) and shaped using an impulse heat sealer. To minimize sticking of samples to the walls during a flushing cycle, the bag was vertically grooved in a 5-mm interval by alternative folding. Two films of 5.5-cm height were fused with the bag vertically along the midline using the heat sealer so as to provide wings for attaching the bag to the chamber plates. The bag was inserted into the movement chamber of the device in a 37 °C environment, and the four wings of the bag were fastened and then attached to the chamber plates using tape. The sample was put into the bag with 5 ml of blocking solution, and 50 μl of antibody was added. Compression and relaxation movements operated by the stepper motor and leadscrew were remotely controlled using a Python script. For an 8-mm-thick swollen sample, the chamber was operated with a travel distance of 10 mm (starting at an 11.25-mm height and compressing to a 1.25-mm height). The travel speed was set to 2 mm s<sup>−1</sup>, and the cycling time intervals were set to 40 s for compression and acceleration and 10 s for relaxation and flushing (by means of upward and downward movements). After antibody incubation for 1 d, the sample was washed with PBST at 37 °C overnight on a gentle shaker. Before RI-matching, the antibody in the sample was fixed by incubating in an antibody fixation solution at 37 °C for 6 h and washing with PBS with azide twice for 2 h each at 37 °C.</p><h3 class="c-article__sub-heading" id="Sec12">RI-matching</h3><p>The whole-area human brain hemisphere slabs of 2-mm thickness, the immunostained 5-mm-thick human sample, the 1.4-mm-thick mouse samples for the compression test and the originally 3-mm-thick mouse samples for the cyclic compression test were optically cleared by RI-matching. The 2-mm-thick slabs were incubated in 100–150 ml of exPROTOS in a 250-ml glass bottle at 37 °C for 1 d, exchanging the medium twice. Other samples, including the immunostained thick sample, the 1.4-mm-thick ELASTicized mouse brain samples and the ELASTicized 3-mm-thick mouse brain samples, were RI-matched using dPROTOS. Samples were incubated at 37 °C according to the following sequences: for the 5-mm-thick human sample, 10 ml of 50% medium (mixed with DI water) for 6 h, 10 ml of 100% medium overnight and fresh 10 ml of medium for 6 h; and for the 1.4-mm-thick samples, 1.5 ml of 50% medium for 2 h and 1.5 ml of 100% medium overnight. Other samples that were microscopically imaged were not RI-matched due to sufficient transparency and/or unnecessity of imaging deep regions. Mouse and human brain tissue samples to measure their size changes were first incubated in dPROTOS at 37 °C for 1 d, and photos were taken. The same procedure was repeated with exPROTOS.</p><h3 class="c-article__sub-heading" id="Sec13">Microscopic imaging of ELASTicized tissues</h3><p>For the antibody delivery speed comparison, two strips of Blu-Tack adhesive were placed on a slide glass, and each of the 1-mm-thick samples was put between the strips. A squared no. 1 coverslip was placed and gently compressed to fit the sample thickness. About 100 μl of PBST was added to prevent the sample from drying. The samples were imaged using the Olympus FV1200MPE laser-scanning confocal microscope with a 20×, 1.0-NA water-immersion objective (XLUMPLFLN 20XW) and a 559-nm diode laser. Central tissue locations were chosen to avoid misinterpretation by lateral diffusion, and longitudinal regions (8:1) were imaged along the whole depth with a constant laser power and gain parameter and then projected to <i>xz</i> planes using Imaris software (Bitplane). For labeling and imaging of the 5-mm-thick human sample, a Blu-Tack adhesive strip was placed on a 60-mm-diameter petri dish (351007; Corning) along the sample boundary, and three Blu-Tack adhesive balls were put along the dish boundary as spacers. The sample incubated in the RI-matching medium was placed on the dish and covered by a WillCo dish with no bubble-trapping. The WillCo dish was gently compressed to match the sample thickness, and the space between the two dishes was filled with the same medium. The gap between the two dishes exposed to the air was sealed with a long Blu-Tack strip. The mounted sample was left at 37 °C for 2 h for RI stabilization. The sample was imaged using the same Olympus microscope with a 10×, 0.6-NA CLARITY-optimized objective (XLPLN10XSVMP, 8.0-mm working distance) and a 559-nm diode laser. The objective was immersed in the same RI-matching medium poured in the WillCo dish, and imaging was started 30 min after installation to stabilize the RI around the objective. A central tissue region was imaged along the whole depth with a constant laser power and gain parameter. The imaged volume was three-dimensionally rendered using the Imaris software, and a video showing axial navigation was generated using ImageJ. The thin sections used for the antibody compatibility test were mounted on slide glasses with Blu-Tack adhesive balls at four corners and covered with no. 1 coverslips. The space near the samples was filled with PBST. Human tissue sections were imaged using the Leica TCS SP8 laser-scanning confocal microscope with a white-light laser source for excitations at 488 and 647 nm. A 63×, 1.3-NA glycerol-immersion objective (HC PL APO CS2) was used for the samples stained with anti-synapsin 1/2 and anti-PSD-95 antibodies, and a 25×, 0.95-NA water-immersion objective (HCX IRAPO L) was used for other samples. Mouse tissue sections were imaged using the Olympus microscope with a 20×, 1.0-NA water-immersion objective.</p><h3 class="c-article__sub-heading" id="Sec14">Fluorescence in situ hybridization (FISH)</h3><p>Thy1-GFP-M mouse SHIELD-ELAST brain hemisphere was used. All clearing steps were performed at 37 °C instead of 45 °C. A SHIELD-ELAST coronal hemisphere tissue with 3-mm thickness was prepared by vibratome-slicing of the hemisphere. The slice was split into 2 mm × 1 cm × 3 mm blocks, and these blocks were subjected to FISH experiments. A block was incubated in the prehybridization buffer containing 2× SSC, 30% formamide, 10% dextran sulfate (average molecular weight, 500 kDa; D8906; MilliporeSigma) and 1 mg ml<sup>−1</sup> denatured/snap-cooled salmon sperm DNA (D7656; MilliporeSigma) at 37 °C for 1 h. Then the FISH probes with 3′ hybridization chain reaction (HCR) initiator overhangs (B1-<i>YFP</i>, 20 nucleotides; B5-<i>vGluT1</i>, 35 nucleotides) were added at final concentration of 1 nM and further incubated at 37 °C overnight<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Park, Y.-G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. Nat. Biotechnol. 37, 73–83 (2019)." href="/articles/s41592-020-0823-y#ref-CR23" id="ref-link-section-d51879e1437">23</a></sup>. The tissue block was washed by 2× SSCT (0.1% triton X-100), 30% formamide buffer at 37 °C for 1 h, three times. Additional 2× SSCT washing for 1 h, two times at RT was performed to remove any residual formamide. The tissue block was then immersed in the amplification buffer containing 2× SSC, 10% dextran sulfate buffer for 1 h. While pre-incubating, aliquots of HCR hairpins (B1-Alexa Fluor 647, B5-Alexa Fluor 568) were heat-denatured at 95 °C for 90 s, and snap-cooled in ice at least for 15 min. Cooled HCR hairpins were added to the amplification buffer at the final concentration of 120 nM and the tissue was incubated at RT overnight. After the HCR process was finished, the tissue was washed by 2× SSCT for 1 h, three times at RT and prepared for imaging by immersing in 2× SSCT. The sample was imaged using the Leica microscope with a 25×, 0.95-NA water-immersion objective.</p><h3 class="c-article__sub-heading" id="Sec15">ELASTicization and labeling of cerebral organoids</h3><p>Organoids were grown according to the protocol by Lancaster and Knoblich<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Lancaster, M. A. &amp; Knoblich, J. A. Generation of cerebral organoids from human pluripotent stem cells. Nat. Protoc. 9, 2329–2340 (2014)." href="/articles/s41592-020-0823-y#ref-CR24" id="ref-link-section-d51879e1449">24</a></sup>, with the addition of dual SMAD inhibition between day 6 and day 9 to drive neural differentiation, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Mellios, N. et al. MeCP2-regulated miRNAs control early human neurogenesis through differential effects on ERK and AKT signaling. Mol. Psychiatr. 23, 1051–1065 (2018)." href="/articles/s41592-020-0823-y#ref-CR25" id="ref-link-section-d51879e1453">25</a></sup>. Organoids were fixed after 35 d of culture with 4% PFA. Organoids were rinsed in PBS three times, and transferred into ice-cold 2% SHIELD solution with 100 mM phosphate buffer (pH 7.2) for 2 d. Organoids were subsequently transferred into 100 mM sodium carbonate buffer (pH 10) and incubated at 37 °C for 1 d. Organoids were extensively washed with PBS then cleared in 0.2 M SDS in 50 mM phosphate buffer (pH 7.3) for 48 h at 55 °C. Organoids were washed with PBST without azide at 37 °C for 1 d, and were incubated in ELAST solution at 4 °C for 1 d. Four organoids were inserted into a polymerization cartridge, and polymerization was performed at 35 °C for 6 h. The organoids were delipidated passively at 37 °C for 3 d, and then washed in PBST at 37 °C for 1 d.</p><p>An ELASTicized organoid was incubated at 37 °C in blocking solution for 3 h using cyclic compression (36-s wait time, 9-s flushing time), then stained using cyclic fourfold compression with 15 μg of DAPI, 10 μg of anti-MAP2 antibody conjugated with Alexa Fluor 594 (677807; BioLegend) and 4 μg of anti-vimentin antibody conjugated with Alexa Fluor 647 (801803; BioLegend) in 1 ml of blocking solution for 12 h at 37 °C, and finally washed with cyclic compression in PBST for 12 h at 37 °C. A ~200-μm-thick depth-wise cross-section was cut from the organoid using two parallel, adjacent steel razor blades, making sure to cut through a laterally central section of the organoid to interrogate the most inaccessible core of the organoid. Depth images of the labeled organoid were acquired using the Leica confocal microscope with ultraviolet (405 nm) and white-light lasers (594 and 647 nm) with a 25×, 0.95-NA water-immersion objective.</p><h3 class="c-article__sub-heading" id="Sec16">Quantification of thinning-related deformation</h3><p>ELASTicized human samples of original 1-mm thickness were stained with anti-GFAP antibody conjugated with Alexa Fluor 594 and tomato lectin conjugated with DyLight 488. The samples were mounted in the same manner as the samples used for the antibody delivery speed comparison and left under the same microscope setting for 0.5–2 h until all noticeable sample movement had ceased. A 100-μm surface depth of each sample was imaged with a 2-μm <i>z</i>-step size using the Olympus microscope with a 20×, 1.0-NA water-immersion objective, a 488-nm argon laser and a 559-nm diode laser. The imaged samples were stretched using the stretching apparatus and left in PBST at 37 °C overnight. The samples were then released from stretching and recovered in PBST at RT for 1–2 h. The samples were mounted, stabilized and imaged in the same way. The GFAP channel was used for quantitative analysis using a method described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. Nat. Biotechnol. 34, 973–981 (2016)." href="/articles/s41592-020-0823-y#ref-CR4" id="ref-link-section-d51879e1471">4</a></sup>. In brief, GFAP filament features of each sample were detected in three dimensions and corresponded between two images obtained before and after stretching. The correspondence information was used to generate a regularly spaced three-dimensional deformation mesh. The deformation-related distance change between each pair of mesh points was measured as an error. For each of the measurement distances, the square root of the average of squared errors was collected from each tissue sample.</p><p>ELASTicized mouse samples in 1.4-mm thickness used for the compression test were imaged before compression and after recovery. Before compression, samples were RI-matched and enhanced green fluorescent protein (EGFP) signals were imaged using the Olympus microscope with a 20×, 1.0-NA water-immersion objective and a 488-nm argon laser. Images were three-dimensionally rendered by the Imaris software for display. Imaged samples were then recovered in PBST and subjected to the compression test while photos were taken. After the test, the samples were RI-matched and imaged again. Obtained EGFP images before and after compression were quantitatively analyzed using the same method for the stretching test with human samples and GFAP images. ELASTicized 3-mm-thick mouse samples were obtained from an ELASTicized brain hemisphere by manual slicing and chopping with a razor blade and used for the cyclic compression test in the same manner. The samples were RI-matched and imaged before and after cycles of 42-s 5–6-fold compression and 12-s release at RT over approximately 1 d. The Olympus microscope was used with a 10×, 0.6-NA CLARITY-optimized objective.</p><h3 class="c-article__sub-heading" id="Sec17">Comparisons of depth-wise immunoreactivity</h3><p>To compare compression versus passive GFAP staining, three 1.5-mm-thick ELASTicized human tissue blocks were pre-incubated passively at 37 °C for 6 h in blocking solution, and then stained using cyclic fivefold compression (40-s wait time, 10-s flushing time) with 10 μg of anti-GFAP antibody conjugated with Alexa Fluor 488 (clone 2E1; 644704; BioLegend) in 2 ml of blocking solution for 6 h at RT. The tissues were then washed passively using PBST for 3 h at 37 °C, after which they were cut vertically in the middle, exposing the cut surfaces. The cut surfaces were passively stained using 1.5 μg of anti-GFAP antibody conjugated with Alexa Fluor 594 (clone GA5; 8152; Cell Signaling Technologies) in 300 μl of PBST for 3 h at 37 °C, and then washed passively in PBST for 1 h at 37 °C. To compare passive versus passive GFAP staining, three 1.5-mm-thick ELASTicized human tissue blocks were cut vertically in the middle, and then pre-incubated passively at 37 °C for 6 h in blocking solution. The cut surfaces were passively stained with 1.5 μg of anti-GFAP antibody conjugated with Alexa Fluor 488 (clone 2E1) and 1.5 μg of anti-GFAP antibody conjugated with Alexa Fluor 594 (clone GA5) in 300 μl of PBST for 3 h at 37 °C, and then washed in PBST for 1 h at 37 °C. ‘Noise’ group images (used to compare signal intensity agreement between noise and the passive 594 channel) were generated by drawing pixel values from a uniform distribution from 0 to 65,535. To compare mouse anti-GFP versus EGFP, an ELASTicized 1-mm-thick Thy1-eGFP mouse coronal hemisphere tissue block was pre-incubated passively at 37 °C for 6 h in blocking solution, and then stained using cyclic fivefold compression (40-s wait time, 10-s flushing time) with 10 μl of chicken anti-GFP antibody (A10262; Invitrogen) in 3 ml of blocking solution for 3 h at 37 °C. After 3 h of cyclic compression washing in blocking solution (with two solution changes) at 37 °C, the sample was secondary-stained using cyclic compression for 3 h at 37 °C with 10 μl of goat anti-chicken IgY antibody conjugated with Alexa Fluor 647 (ab150175; Abcam) in 3 ml of blocking solution, followed by cyclic compression washing in PBST for 1 h at 37 °C. All samples’ cut surfaces were then imaged using a 20×, 0.5-NA water-immersion objective with the Leica TCS SP8 confocal microscope with a white-light laser source for 488-nm and 594-nm wavelengths.</p><p>Each <i>z</i>-stack image was post-processed with histogram equalization (using the stack histogram, 0.3% pixels saturated) using ImageJ. The depth-wise signal intensity agreement (<i>A</i><sub><i>k</i></sub>) of each <i>yz</i>-plane image optical section at a given axial position <i>k</i> was computed with:</p><div id="Eque" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$A_k = \frac{1}{{N + M}}\mathop {\sum}\limits_{i = 1}^N {\mathop {\sum}\limits_{j = 1}^M {1 - \frac{{|I_{488,ijk} - I_{594,ijk}|}}{{I_{594,ijk}}}} }$$</span></div></div><p>where <i>N</i> and <i>M</i> are the number of pixels in the two lateral dimensions of each image plane, and <i>I</i><sub><i>λ</i>,<i>ijk</i></sub> represents the signal intensity of the <i>λ</i> wavelength image channel at lateral pixel position (<i>i</i>,<i>j</i>) and axial pixel position <i>k</i>. The depth-wise coefficient of variation (CoV) was computed with:</p><div id="Equf" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathrm{CoV}} = \frac{\sigma }{{\bar A}}$$</span></div></div><p>where the mean and standard deviation are computed by:</p><div id="Equg" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\bar A = \frac{1}{Z}\mathop {\sum}\limits_{k = 1}^Z {A_k}$$</span></div></div><div id="Equh" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\sigma = \sqrt {\frac{1}{Z}\mathop {\sum }\limits_{k = 1}^Z \left( {A_k - \bar A} \right)^2}$$</span></div></div><p>where <i>Z</i> is the number of pixels in the depth-wise direction. The upper and lower bounds of the 95% confidence intervals were computed with ±2 s.d. from the depth-wise mean signal agreement between the three samples in each group. To compute the statistical significance between each group, two-tailed Student’s <i>t</i>-tests were run using the depth-wise mean signal intensity agreements of each group as sample observations. All analyses were performed using custom scripts in numpy, scipy and matplotlib packages in Python 3.6 (<a href="https://www.python.org/">https://www.python.org/</a>).</p><h3 class="c-article__sub-heading" id="Sec18">Downstream histology of ELASTicized tissue</h3><p>A block (approximate lateral dimensions of 2 × 2 mm<sup>2</sup>) from an ELASTicized 1-mm-thick mouse brain hemisphere slice was first immunostained as follows: blocking at 37 °C for 6 h, 6 μg of rabbit anti-NeuN antibody (ab104225; Abcam) in 500 μl of blocking solution at 37 °C for 2 d, washing with PBST three times at 37 °C overnight, 12 μg of donkey anti-rabbit IgG antibody conjugated with Alexa Fluor Plus 594 (A32754; Life Technologies) in 500 μl of PBST at 37 °C overnight and washing with PBST three times at 37 °C for 8 h. A 200-μm-thickness volume of the stained sample was imaged with the Zeiss LSM 780 confocal microscope with a 10×, 0.45-NA dry objective and a 561-nm diode laser.</p><p>After confocal fluorescent microscopic imaging, the sample was cryopreserved in 15% sucrose (S0389; MilliporeSigma) in PBS until sinking followed by 30% sucrose in PBS overnight. Sections of varying thicknesses from 3 to 30 μm (original thickness) were obtained by cryosectioning with the Leica CM1850. Originally 10-μm-thick sections were chosen for the following histological staining, placed on slide glasses, spread with a paint brush and air-dried. The sections were stained as follows: Harris hematoxylin solution (20022; Muto Pure Chemicals) for 10 min, washing carefully with DI water by pipetting, a drop of 1% hydrochloric acid (7647-01-0; Junsei Chemical), washing with DI water and 0.5% eosin Y solution (102439; Merck) for 2 min. The sections were then dehydrated with a series of ethanol solutions, dipped in xylene and mounted on slide glasses with a Dako mounting medium (CS70330; Agilent Technologies). The samples were imaged using the Leica DM6000 B bright-field microscope with a 2.5×, 0.07-NA dry objective.</p><h3 class="c-article__sub-heading" id="Sec19">Statistics and reproducibility</h3><p>Sample numbers and replication types are described in figure legends, where all replicates are either different hydrogels or different tissue samples. For the thickness measurement of hydrogels, the thicknesses of four gels were measured before and after stretching and analyzed using a two-tailed paired <i>t</i>-test. For the comparison of diffusions in hydrogels, eight gels were randomly assigned to either control or stretching group. An effective diffusion coefficient was computed from each gel, and the resulting diffusion timescales were compared using a two-tailed unpaired <i>t</i>-test. For the thickness measurement of ELASTicized human brain samples, ten samples were collected and their thicknesses were measured. The samples were then split into two groups of stretching and contraction-only so as to have average thicknesses as close as possible between the two groups. Thicknesses of stretched samples were measured after stretching and then after contraction, and thicknesses of contracted samples were measured after contraction. One-way analysis of variance was used to compare the thicknesses among the four groups. A post hoc Tukey’s test was performed to compare thicknesses between each pair of groups. Five, five and four samples were prepared for deformation analysis with stretching, manual compression and cyclic compression, respectively, and images obtained before and after stretching or compression were used for error quantification. Tissue size changes by ELASTicization and RI-matching with different media were measured with six mouse samples and four human samples. A tissue size of each sample was estimated as a square root of the product of two crossing lines connecting representative points. The margins of surrounding gels were instead measured for transparent human samples after RI-matching. For the compressed area measurement experiment, four samples were tested. Most of the other experiments, such as the antibody validation, the antibody delivery speed comparison, stretching and RI-matching of large-area slabs, compression of thick samples, depth-wise comparison of EGFP signal and anti-GFP immunoreactivity by cyclic compression, and downstream histology, were carried out at least twice with the same results. The thick-tissue immunolabeling with a 5-mm-thick human brain sample, compression of various mouse organs and immunolabeling of a cerebral organoid were performed once.</p><h3 class="c-article__sub-heading" id="Sec20">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41592-020-0823-y#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                        

                        <section aria-labelledby="data-availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
            
            <p>All data supporting the findings of this study are included in figures and videos as representative images or data points in the plots. Additional images other than the representative images are available from the corresponding author upon reasonable request. Resources that may help enable general users to establish the methodology are freely available online (<a href="http://www.chunglabresources.org">http://www.chunglabresources.org</a>).</p>
          </div></div></section><section aria-labelledby="code-availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
            
            <p>The custom code used in this study is available from the corresponding author upon reasonable request.</p>
          </div></div></section><section aria-labelledby="Bib1"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. Chung, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="Chung, K. et al. Structural and molecular interrogation of intact biological systems. Nature 497, 332–337 (201" /><span class="c-article-references__counter">1.</span><p class="c-article-references__text" id="ref-CR1">Chung, K. et al. Structural and molecular interrogation of intact biological systems. <i>Nature</i> <b>497</b>, 332–337 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXlslSkurg%253D" aria-label="View reference 1 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnature12107" aria-label="View reference 1">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20and%20molecular%20interrogation%20of%20intact%20biological%20systems&amp;journal=Nature&amp;volume=497&amp;pages=332-337&amp;publication_year=2013&amp;author=Chung%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="F. Chen, PW. Tillberg, ES. Boyden, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Chen, F., Tillberg, P. W. &amp; Boyden, E. S. Expansion microscopy. Science 347, 543–548 (2015)." /><span class="c-article-references__counter">2.</span><p class="c-article-references__text" id="ref-CR2">Chen, F., Tillberg, P. W. &amp; Boyden, E. S. Expansion microscopy. <i>Science</i> <b>347</b>, 543–548 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhsV2ns7w%253D" aria-label="View reference 2 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1260088" aria-label="View reference 2">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 2 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Expansion%20microscopy&amp;journal=Science&amp;volume=347&amp;pages=543-548&amp;publication_year=2015&amp;author=Chen%2CF&amp;author=Tillberg%2CPW&amp;author=Boyden%2CES">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="EL. Sylwestrak, P. Rajasethupathy, MA. Wright, A. Jaffe, K. Deisseroth, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Sylwestrak, E. L., Rajasethupathy, P., Wright, M. A., Jaffe, A. &amp; Deisseroth, K. Multiplexed intact-tissue tra" /><span class="c-article-references__counter">3.</span><p class="c-article-references__text" id="ref-CR3">Sylwestrak, E. L., Rajasethupathy, P., Wright, M. A., Jaffe, A. &amp; Deisseroth, K. Multiplexed intact-tissue transcriptional analysis at cellular resolution. <i>Cell</i> <b>164</b>, 792–804 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XisFOgtrw%253D" aria-label="View reference 3 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2016.01.038" aria-label="View reference 3">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 3 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20intact-tissue%20transcriptional%20analysis%20at%20cellular%20resolution&amp;journal=Cell&amp;volume=164&amp;pages=792-804&amp;publication_year=2016&amp;author=Sylwestrak%2CEL&amp;author=Rajasethupathy%2CP&amp;author=Wright%2CMA&amp;author=Jaffe%2CA&amp;author=Deisseroth%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Ku, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in s" /><span class="c-article-references__counter">4.</span><p class="c-article-references__text" id="ref-CR4">Ku, T. et al. Multiplexed and scalable super-resolution imaging of three-dimensional protein localization in size-adjustable tissues. <i>Nat. Biotechnol.</i> <b>34</b>, 973–981 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1KmurbE" aria-label="View reference 4 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.3641" aria-label="View reference 4">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20and%20scalable%20super-resolution%20imaging%20of%20three-dimensional%20protein%20localization%20in%20size-adjustable%20tissues&amp;journal=Nat.%20Biotechnol.&amp;volume=34&amp;pages=973-981&amp;publication_year=2016&amp;author=Ku%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Science 361," /><span class="c-article-references__counter">5.</span><p class="c-article-references__text" id="ref-CR5">Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. <i>Science</i> <b>361</b>, eaat5691 (2018).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.aat5691" aria-label="View reference 5">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 5 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Three-dimensional%20intact-tissue%20sequencing%20of%20single-cell%20transcriptional%20states&amp;journal=Science&amp;volume=361&amp;publication_year=2018&amp;author=Wang%2CX">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="HR. Ueda, " /><meta itemprop="datePublished" content="2020" /><meta itemprop="headline" content="Ueda, H. R. et al. Tissue clearing and its applications in neuroscience. Nat. Rev. Neurosci. 21, 61–79 (2020)." /><span class="c-article-references__counter">6.</span><p class="c-article-references__text" id="ref-CR6">Ueda, H. R. et al. Tissue clearing and its applications in neuroscience. <i>Nat. Rev. Neurosci.</i> <b>21</b>, 61–79 (2020).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXjtlSntw%253D%253D" aria-label="View reference 6 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fs41583-019-0250-1" aria-label="View reference 6">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Tissue%20clearing%20and%20its%20applications%20in%20neuroscience&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=21&amp;pages=61-79&amp;publication_year=2020&amp;author=Ueda%2CHR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="F. Chen, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Chen, F. et al. Nanoscale imaging of RNA with expansion microscopy. Nat. Methods 13, 679–684 (2016)." /><span class="c-article-references__counter">7.</span><p class="c-article-references__text" id="ref-CR7">Chen, F. et al. Nanoscale imaging of RNA with expansion microscopy. <i>Nat. Methods</i> <b>13</b>, 679–684 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtFWjs7%252FJ" aria-label="View reference 7 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth.3899" aria-label="View reference 7">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Nanoscale%20imaging%20of%20RNA%20with%20expansion%20microscopy&amp;journal=Nat.%20Methods&amp;volume=13&amp;pages=679-684&amp;publication_year=2016&amp;author=Chen%2CF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H-E. Park, " /><meta itemprop="datePublished" content="2019" /><meta itemprop="headline" content="Park, H.-E. et al. Scalable and isotropic expansion of tissues with simply tunable expansion ratio. Adv. Sci. " /><span class="c-article-references__counter">8.</span><p class="c-article-references__text" id="ref-CR8">Park, H.-E. et al. Scalable and isotropic expansion of tissues with simply tunable expansion ratio. <i>Adv. Sci.</i> <b>6</b>, 1901673 (2019).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXisVartrfN" aria-label="View reference 8 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1002%2Fadvs.201901673" aria-label="View reference 8">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 8 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Scalable%20and%20isotropic%20expansion%20of%20tissues%20with%20simply%20tunable%20expansion%20ratio&amp;journal=Adv.%20Sci.&amp;volume=6&amp;publication_year=2019&amp;author=Park%2CH-E">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="YS. Zhang, A. Khademhosseini, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Zhang, Y. S. &amp; Khademhosseini, A. Advances in engineering hydrogels. Science 356, eaaf3627 (2017)." /><span class="c-article-references__counter">9.</span><p class="c-article-references__text" id="ref-CR9">Zhang, Y. S. &amp; Khademhosseini, A. Advances in engineering hydrogels. <i>Science</i> <b>356</b>, eaaf3627 (2017).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.aaf3627" aria-label="View reference 9">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20engineering%20hydrogels&amp;journal=Science&amp;volume=356&amp;publication_year=2017&amp;author=Zhang%2CYS&amp;author=Khademhosseini%2CA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Okumura, K. Ito, " /><meta itemprop="datePublished" content="2001" /><meta itemprop="headline" content="Okumura, Y. &amp; Ito, K. The polyrotaxane gel: a topological gel by figure-of-eight cross-links. Adv. Mater. 13, " /><span class="c-article-references__counter">10.</span><p class="c-article-references__text" id="ref-CR10">Okumura, Y. &amp; Ito, K. The polyrotaxane gel: a topological gel by figure-of-eight cross-links. <i>Adv. Mater.</i> <b>13</b>, 485–487 (2001).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXjtVWktb4%253D" aria-label="View reference 10 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1002%2F1521-4095%28200104%2913%3A7%3C485%3A%3AAID-ADMA485%3E3.0.CO%3B2-T" aria-label="View reference 10">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 10 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20polyrotaxane%20gel%3A%20a%20topological%20gel%20by%20figure-of-eight%20cross-links&amp;journal=Adv.%20Mater.&amp;volume=13&amp;pages=485-487&amp;publication_year=2001&amp;author=Okumura%2CY&amp;author=Ito%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MJ. Zhong, R. Wang, K. Kawamoto, BD. Olsen, JA. Johnson, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Zhong, M. J., Wang, R., Kawamoto, K., Olsen, B. D. &amp; Johnson, J. A. Quantifying the impact of molecular defect" /><span class="c-article-references__counter">11.</span><p class="c-article-references__text" id="ref-CR11">Zhong, M. J., Wang, R., Kawamoto, K., Olsen, B. D. &amp; Johnson, J. A. Quantifying the impact of molecular defects on polymer network elasticity. <i>Science</i> <b>353</b>, 1264–1268 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhsV2ksbvE" aria-label="View reference 11 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.aag0184" aria-label="View reference 11">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20the%20impact%20of%20molecular%20defects%20on%20polymer%20network%20elasticity&amp;journal=Science&amp;volume=353&amp;pages=1264-1268&amp;publication_year=2016&amp;author=Zhong%2CMJ&amp;author=Wang%2CR&amp;author=Kawamoto%2CK&amp;author=Olsen%2CBD&amp;author=Johnson%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JP. Gong, Y. Katsuyama, T. Kurokawa, Y. Osada, " /><meta itemprop="datePublished" content="2003" /><meta itemprop="headline" content="Gong, J. P., Katsuyama, Y., Kurokawa, T. &amp; Osada, Y. Double-network hydrogels with extremely high mechanical s" /><span class="c-article-references__counter">12.</span><p class="c-article-references__text" id="ref-CR12">Gong, J. P., Katsuyama, Y., Kurokawa, T. &amp; Osada, Y. Double-network hydrogels with extremely high mechanical strength. <i>Adv. Mater.</i> <b>15</b>, 1155–1158 (2003).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXmtFemu7k%253D" aria-label="View reference 12 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1002%2Fadma.200304907" aria-label="View reference 12">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Double-network%20hydrogels%20with%20extremely%20high%20mechanical%20strength&amp;journal=Adv.%20Mater.&amp;volume=15&amp;pages=1155-1158&amp;publication_year=2003&amp;author=Gong%2CJP&amp;author=Katsuyama%2CY&amp;author=Kurokawa%2CT&amp;author=Osada%2CY">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J-Y. Sun, " /><meta itemprop="datePublished" content="2012" /><meta itemprop="headline" content="Sun, J.-Y. et al. Highly stretchable and tough hydrogels. Nature 489, 133–136 (2012)." /><span class="c-article-references__counter">13.</span><p class="c-article-references__text" id="ref-CR13">Sun, J.-Y. et al. Highly stretchable and tough hydrogels. <i>Nature</i> <b>489</b>, 133–136 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XhtlGnt73N" aria-label="View reference 13 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnature11409" aria-label="View reference 13">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 13 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20stretchable%20and%20tough%20hydrogels&amp;journal=Nature&amp;volume=489&amp;pages=133-136&amp;publication_year=2012&amp;author=Sun%2CJ-Y">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JP. Gong, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Gong, J. P. Materials both tough and soft. Science 344, 161–162 (2014)." /><span class="c-article-references__counter">14.</span><p class="c-article-references__text" id="ref-CR14">Gong, J. P. Materials both tough and soft. <i>Science</i> <b>344</b>, 161–162 (2014).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXosFSntrc%253D" aria-label="View reference 14 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1252389" aria-label="View reference 14">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Materials%20both%20tough%20and%20soft&amp;journal=Science&amp;volume=344&amp;pages=161-162&amp;publication_year=2014&amp;author=Gong%2CJP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="RC. Ball, M. Doi, SF. Edwards, M. Warner, " /><meta itemprop="datePublished" content="1981" /><meta itemprop="headline" content="Ball, R. C., Doi, M., Edwards, S. F. &amp; Warner, M. Elasticity of entangled networks. Polymer 22, 1010–1018 (198" /><span class="c-article-references__counter">15.</span><p class="c-article-references__text" id="ref-CR15">Ball, R. C., Doi, M., Edwards, S. F. &amp; Warner, M. Elasticity of entangled networks. <i>Polymer</i> <b>22</b>, 1010–1018 (1981).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaL38Xnt12j" aria-label="View reference 15 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2F0032-3861%2881%2990284-6" aria-label="View reference 15">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Elasticity%20of%20entangled%20networks&amp;journal=Polymer&amp;volume=22&amp;pages=1010-1018&amp;publication_year=1981&amp;author=Ball%2CRC&amp;author=Doi%2CM&amp;author=Edwards%2CSF&amp;author=Warner%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Hama, " /><meta itemprop="datePublished" content="2011" /><meta itemprop="headline" content="Hama, H. et al. Scale: a chemical approach for fluorescence imaging and reconstruction of transparent mouse br" /><span class="c-article-references__counter">16.</span><p class="c-article-references__text" id="ref-CR16">Hama, H. et al. Sca<i>l</i>e: a chemical approach for fluorescence imaging and reconstruction of transparent mouse brain. <i>Nat. Neurosci.</i> <b>14</b>, 1481–1488 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtV2gtr7L" aria-label="View reference 16 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnn.2928" aria-label="View reference 16">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Scale%3A%20a%20chemical%20approach%20for%20fluorescence%20imaging%20and%20reconstruction%20of%20transparent%20mouse%20brain&amp;journal=Nat.%20Neurosci.&amp;volume=14&amp;pages=1481-1488&amp;publication_year=2011&amp;author=Hama%2CH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M-T. Ke, S. Fujimoto, T. Imai, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="Ke, M.-T., Fujimoto, S. &amp; Imai, T. SeeDB: a simple and morphology-preserving optical clearing agent for neuron" /><span class="c-article-references__counter">17.</span><p class="c-article-references__text" id="ref-CR17">Ke, M.-T., Fujimoto, S. &amp; Imai, T. SeeDB: a simple and morphology-preserving optical clearing agent for neuronal circuit reconstruction. <i>Nat. Neurosci.</i> <b>16</b>, 1154–1161 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXpvVyrtLw%253D" aria-label="View reference 17 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnn.3447" aria-label="View reference 17">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 17 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=SeeDB%3A%20a%20simple%20and%20morphology-preserving%20optical%20clearing%20agent%20for%20neuronal%20circuit%20reconstruction&amp;journal=Nat.%20Neurosci.&amp;volume=16&amp;pages=1154-1161&amp;publication_year=2013&amp;author=Ke%2CM-T&amp;author=Fujimoto%2CS&amp;author=Imai%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Pan, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Pan, C. et al. Shrinkage-mediated imaging of entire organs and organisms using uDISCO. Nat. Methods 13, 859–86" /><span class="c-article-references__counter">18.</span><p class="c-article-references__text" id="ref-CR18">Pan, C. et al. Shrinkage-mediated imaging of entire organs and organisms using uDISCO. <i>Nat. Methods</i> <b>13</b>, 859–867 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtlyqsbnK" aria-label="View reference 18 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth.3964" aria-label="View reference 18">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 18 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Shrinkage-mediated%20imaging%20of%20entire%20organs%20and%20organisms%20using%20uDISCO&amp;journal=Nat.%20Methods&amp;volume=13&amp;pages=859-867&amp;publication_year=2016&amp;author=Pan%2CC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="TC. Murakami, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Murakami, T. C. et al. A three-dimensional single-cell-resolution whole-brain atlas using CUBIC-X expansion mi" /><span class="c-article-references__counter">19.</span><p class="c-article-references__text" id="ref-CR19">Murakami, T. C. et al. A three-dimensional single-cell-resolution whole-brain atlas using CUBIC-X expansion microscopy and tissue clearing. <i>Nat. Neurosci.</i> <b>21</b>, 625–637 (2018).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXltVyhsbY%253D" aria-label="View reference 19 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fs41593-018-0109-1" aria-label="View reference 19">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 19 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20three-dimensional%20single-cell-resolution%20whole-brain%20atlas%20using%20CUBIC-X%20expansion%20microscopy%20and%20tissue%20clearing&amp;journal=Nat.%20Neurosci.&amp;volume=21&amp;pages=625-637&amp;publication_year=2018&amp;author=Murakami%2CTC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="E. Murray, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Murray, E. et al. Simple, scalable proteomic imaging for high-dimensional profiling of intact systems. Cell 16" /><span class="c-article-references__counter">20.</span><p class="c-article-references__text" id="ref-CR20">Murray, E. et al. Simple, scalable proteomic imaging for high-dimensional profiling of intact systems. <i>Cell</i> <b>163</b>, 1500–1514 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhvFKksb%252FJ" aria-label="View reference 20 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2015.11.025" aria-label="View reference 20">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Simple%2C%20scalable%20proteomic%20imaging%20for%20high-dimensional%20profiling%20of%20intact%20systems&amp;journal=Cell&amp;volume=163&amp;pages=1500-1514&amp;publication_year=2015&amp;author=Murray%2CE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Zhao, " /><meta itemprop="datePublished" content="2020" /><meta itemprop="headline" content="Zhao, S. et al. Cellular and molecular probing of intact human organs. Cell 180, 796–812 (2020)." /><span class="c-article-references__counter">21.</span><p class="c-article-references__text" id="ref-CR21">Zhao, S. et al. Cellular and molecular probing of intact human organs. <i>Cell</i> <b>180</b>, 796–812 (2020).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXjsVOmtbY%253D" aria-label="View reference 21 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2020.01.030" aria-label="View reference 21">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 21 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Cellular%20and%20molecular%20probing%20of%20intact%20human%20organs&amp;journal=Cell&amp;volume=180&amp;pages=796-812&amp;publication_year=2020&amp;author=Zhao%2CS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="SI. Kubota, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Kubota, S. I. et al. Whole-body profiling of cancer metastasis with single-cell resolution. Cell Rep. 20, 236–" /><span class="c-article-references__counter">22.</span><p class="c-article-references__text" id="ref-CR22">Kubota, S. I. et al. Whole-body profiling of cancer metastasis with single-cell resolution. <i>Cell Rep.</i> <b>20</b>, 236–250 (2017).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhtFartLfJ" aria-label="View reference 22 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.celrep.2017.06.010" aria-label="View reference 22">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 22 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-body%20profiling%20of%20cancer%20metastasis%20with%20single-cell%20resolution&amp;journal=Cell%20Rep.&amp;volume=20&amp;pages=236-250&amp;publication_year=2017&amp;author=Kubota%2CSI">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y-G. Park, " /><meta itemprop="datePublished" content="2019" /><meta itemprop="headline" content="Park, Y.-G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. Nat. Bio" /><span class="c-article-references__counter">23.</span><p class="c-article-references__text" id="ref-CR23">Park, Y.-G. et al. Protection of tissue physicochemical properties using polyfunctional crosslinkers. <i>Nat. Biotechnol.</i> <b>37</b>, 73–83 (2019).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXisFejsrnL" aria-label="View reference 23 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.4281" aria-label="View reference 23">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 23 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Protection%20of%20tissue%20physicochemical%20properties%20using%20polyfunctional%20crosslinkers&amp;journal=Nat.%20Biotechnol.&amp;volume=37&amp;pages=73-83&amp;publication_year=2019&amp;author=Park%2CY-G">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MA. Lancaster, JA. Knoblich, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Lancaster, M. A. &amp; Knoblich, J. A. Generation of cerebral organoids from human pluripotent stem cells. Nat. Pr" /><span class="c-article-references__counter">24.</span><p class="c-article-references__text" id="ref-CR24">Lancaster, M. A. &amp; Knoblich, J. A. Generation of cerebral organoids from human pluripotent stem cells. <i>Nat. Protoc.</i> <b>9</b>, 2329–2340 (2014).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhsVyhu7%252FO" aria-label="View reference 24 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnprot.2014.158" aria-label="View reference 24">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 24 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20of%20cerebral%20organoids%20from%20human%20pluripotent%20stem%20cells&amp;journal=Nat.%20Protoc.&amp;volume=9&amp;pages=2329-2340&amp;publication_year=2014&amp;author=Lancaster%2CMA&amp;author=Knoblich%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="N. Mellios, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Mellios, N. et al. MeCP2-regulated miRNAs control early human neurogenesis through differential effects on ERK" /><span class="c-article-references__counter">25.</span><p class="c-article-references__text" id="ref-CR25">Mellios, N. et al. MeCP2-regulated miRNAs control early human neurogenesis through differential effects on ERK and AKT signaling. <i>Mol. Psychiatr.</i> <b>23</b>, 1051–1065 (2018).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXmtlSiu7k%253D" aria-label="View reference 25 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fmp.2017.86" aria-label="View reference 25">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 25 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=MeCP2-regulated%20miRNAs%20control%20early%20human%20neurogenesis%20through%20differential%20effects%20on%20ERK%20and%20AKT%20signaling&amp;journal=Mol.%20Psychiatr.&amp;volume=23&amp;pages=1051-1065&amp;publication_year=2018&amp;author=Mellios%2CN">
                        Google Scholar</a></li></ul></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-category="article body" data-track-label="link" href="/articles/s41592-020-0823-y-references.ris">Download references<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank the entire Chung Laboratory for support and discussions. We thank S.-C. Chen and J. Wang for the development of the vibrating-blade microtome, and D. H. Yun for its operation. We also thank K. Xie for mouse sample preparation, Y. Tian for the initial diffusion modeling work and R. Thorndike-Breeze for comments on improving the manuscript. We thank H. Jung and H. Kim for providing the histology facility and helping the H&amp;E experiment. K.C. was supported by the Burroughs Wellcome Fund Career Awards at the Scientific Interface, the Searle Scholars Program, the Packard Award in Science and Engineering, the NARSAD Young Investigator Award, the Institute for Basic Science grant no. IBS-R026-D1 and the McKnight Foundation Technology Award. W.G. was supported by the National Science Foundation Graduate Research Fellowship under grant no. 1122374. A.A. was supported by the JPB Foundation (Picower Fellowship). M.P.F. was partially supported by NIA grant no. P50 AG005134. This work was supported by the JPB Foundation (grant nos. PIIF and PNDRF), the NCSOFT Cultural Foundation, and the NIH (grant nos. 1-DP2-ES027992, U01MH117072).</p></div></div></section><section aria-labelledby="author-information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff10"><ul class="c-article-author-information__authors-list"><li class="c-article-author-affiliation__authors-item">Taeyun Ku</li></ul><p class="js-present-address">Present address: Graduate School of Medical Science and Engineering, KAIST, Daejeon, Republic of Korea</p></li><li class="c-article-author-information__item" id="nAff11"><ul class="c-article-author-information__authors-list"><li class="c-article-author-affiliation__authors-item">Chang Ho Sohn</li></ul><p class="js-present-address">Present address: Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea</p></li><li class="c-article-author-information__item" id="nAff12"><ul class="c-article-author-information__authors-list"><li class="c-article-author-affiliation__authors-item">Chang Ho Sohn</li></ul><p class="js-present-address">Present address: Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><span class="c-article-author-affiliation__address">Institute for Medical Engineering and Science, Massachusetts Institute of Technology (MIT), Cambridge, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Taeyun Ku</li><li class="c-article-author-affiliation__authors-item">, Nicholas B. Evans</li><li class="c-article-author-affiliation__authors-item">, Chang Ho Sohn</li><li class="c-article-author-affiliation__authors-item">, Alexandre Albanese</li><li class="c-article-author-affiliation__authors-item"> &amp; Kwanghun Chung</li></ul></li><li id="Aff2"><span class="c-article-author-affiliation__address">Picower Institute for Learning and Memory, MIT, Cambridge, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Taeyun Ku</li><li class="c-article-author-affiliation__authors-item">, Nicholas B. Evans</li><li class="c-article-author-affiliation__authors-item">, Chang Ho Sohn</li><li class="c-article-author-affiliation__authors-item"> &amp; Kwanghun Chung</li></ul></li><li id="Aff3"><span class="c-article-author-affiliation__address">Department of Chemical Engineering, MIT, Cambridge, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Webster Guan</li><li class="c-article-author-affiliation__authors-item"> &amp; Kwanghun Chung</li></ul></li><li id="Aff4"><span class="c-article-author-affiliation__address">Graduate School of Medical Science and Engineering, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Joon-Goon Kim</li></ul></li><li id="Aff5"><span class="c-article-author-affiliation__address">C.S. Kubik Laboratory for Neuropathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Matthew P. Frosch</li></ul></li><li id="Aff6"><span class="c-article-author-affiliation__address">Department of Brain and Cognitive Sciences, MIT, Cambridge, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kwanghun Chung</li></ul></li><li id="Aff7"><span class="c-article-author-affiliation__address">Broad Institute of Harvard University and MIT, Cambridge, MA, USA</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kwanghun Chung</li></ul></li><li id="Aff8"><span class="c-article-author-affiliation__address">Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kwanghun Chung</li></ul></li><li id="Aff9"><span class="c-article-author-affiliation__address">Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kwanghun Chung</li></ul></li><li id="Aff11"><span class="c-article-author-affiliation__address">Center for Nanomedicine, Institute for Basic Science (IBS), Seoul, Republic of Korea</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Chang Ho Sohn</li></ul></li><li id="Aff12"><span class="c-article-author-affiliation__address">Nano Biomedical Engineering (Nano BME) Graduate Program, Yonsei-IBS Institute, Yonsei University, Seoul, Republic of Korea</span><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Chang Ho Sohn</li></ul></li></ol><div class="js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-1"><span class="c-article-authors-search__title u-h3 js-search-name">Taeyun Ku</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Taeyun+Ku&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Taeyun+Ku" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Taeyun+Ku%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-2"><span class="c-article-authors-search__title u-h3 js-search-name">Webster Guan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Webster+Guan&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Webster+Guan" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Webster+Guan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-3"><span class="c-article-authors-search__title u-h3 js-search-name">Nicholas B. Evans</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Nicholas B.+Evans&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicholas B.+Evans" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicholas B.+Evans%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-4"><span class="c-article-authors-search__title u-h3 js-search-name">Chang Ho Sohn</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Chang Ho+Sohn&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chang Ho+Sohn" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chang Ho+Sohn%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-5"><span class="c-article-authors-search__title u-h3 js-search-name">Alexandre Albanese</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Alexandre+Albanese&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alexandre+Albanese" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alexandre+Albanese%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-6"><span class="c-article-authors-search__title u-h3 js-search-name">Joon-Goon Kim</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Joon-Goon+Kim&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joon-Goon+Kim" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joon-Goon+Kim%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-7"><span class="c-article-authors-search__title u-h3 js-search-name">Matthew P. Frosch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Matthew P.+Frosch&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matthew P.+Frosch" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matthew P.+Frosch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li><li id="auth-8"><span class="c-article-authors-search__title u-h3 js-search-name">Kwanghun Chung</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=&#34;Kwanghun+Chung&#34;" class="c-article-button" data-track="click" data-track-category="Article body" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><span class="search-in-title-js">You can also search for this author in </span><ul class="c-article-identifiers"><li class="c-article-identifiers__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kwanghun+Chung" data-track="click" data-track-category="Article body" data-track-action="author link - pubmed" data-track-label="link">PubMed</a></li><li class="c-article-identifiers__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kwanghun+Chung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-category="Article body" data-track-action="author link - scholar" data-track-label="link">
                                Google Scholar
                            </a></li></ul></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>T.K. and K.C. designed the experiments and wrote the paper with input from other authors. T.K. developed ELAST and conducted hydrogel experiments, evaluation of mechanical properties and tissue deformation, antibody screening and delivery tests, and the deep-tissue labeling experiment. W.G. modeled the diffusion profiles in hydrogels and conducted the cyclic compression experiment to assess depth-wise immunostaining quality. N.B.E., T.K. and W.G. developed and operated the cyclic compression device. N.B.E. formulated the RI-matching media. C.H.S. and T.K. established the mouse SHIELD protocol for ELAST. C.H.S. conducted the in situ hybridization experiment. A.A. and W.G. conducted the organoid experiment. J.-G.K. and T.K. conducted the downstream histology experiment. M.P.F. provided the human brain tissue specimens. K.C. supervised all aspects of the work.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p>Correspondence to
                <a id="corresp-c1" rel="nofollow" href="/articles/s41592-020-0823-y/email/correspondent/c1/new">Kwanghun Chung</a>.</p></div></div></section><section aria-labelledby="ethics"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
            
              <h3 class="c-article__sub-heading">Competing interests</h3>
              <p>The ELAST concepts and applications are covered in a pending patent application owned by MIT (K.C. and T.K.). K.C. is a cofounder of LifeCanvas Technologies, a startup that provides solutions for 3D tissue processing.</p>
            
          </div></div></section><section aria-labelledby="additional-information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> Nina Vogt was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section aria-labelledby="Sec22"><div class="c-article-section" id="Sec22-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec22">Extended data</h2><div class="c-article-section__content" id="Sec22-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="/articles/s41592-020-0823-y/figures/4" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig4_ESM.jpg">Extended Data Fig. 1 Dense entangled pAAm hydrogels that are elastic, tough, and stretchable.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, High-concentration acrylamide (AAm) solution forms an entangled hydrogel in the absence of cross-linker, <i>N</i>, <span class="mathjax-tex">\(N^\prime\)</span>-methylenebisacrylamide (MBAA). <b>b,</b> A compression test on 40% (wt/vol) pAAm hydrogels with (upper panels) and without (lower panels) 0.5% (wt/vol) MBAA. The gels were prepared with 0.005% (wt/vol) VA-044 and nitrogen gas-bubbling. Gels with 9-mm thickness were compressed to 1 mm. <i>λ</i>, compression strain. <b>c,</b> A demonstration of stretching a 36% (wt/vol) entangled pAAm gel (no MBAA and 0.005% (wt/vol) VA-044 with nitrogen gas-bubbling). Representative images are shown from experiments repeated two times (<b>a</b> and <b>b</b>) and three times (<b>c</b>) with similar results.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="/articles/s41592-020-0823-y/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig5_ESM.jpg">Extended Data Fig. 2 ELASTicization and compression of various mouse organs and a cerebral organoid.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a,</b> Compression of ELASTicized mouse organs. Samples were manually compressed between two glass plates. The experiment was not repeated. <b>b,</b> Compression of an ELASTicized cerebral organoid. Scale bars, 5 mm. Experiment was repeated two times with similar results.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="/articles/s41592-020-0823-y/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_Fig6_ESM.jpg">Extended Data Fig. 3 Downstream histology of ELASTicized tissue.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a,</b> An ELASTicized 1-mm-thick mouse brain slice block was immunolabeled with an anti-NeuN antibody and imaged by confocal fluorescence microscopy. <b>b,</b> The same sample in <b>a</b> was then cryo-sectioned into 10-μm-thick sections. One section was histologically stained with hematoxylin and eosin with following a series of dehydration steps using ethanol solutions and xylene. The image was obtained by a bright-field microscope. Note that the image planes were not intended to match. Scale bar (adjusted to match the original tissue dimensions), 200 μm. The hematoxylin and eosin staining experiment in <b>b</b> was repeated six times with similar results using two NeuN-stained samples as in <b>a</b>.</p></div></div></div></div></div></section><section aria-labelledby="Sec23"><div class="c-article-section" id="Sec23-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec23">Supplementary information</h2><div class="c-article-section__content" id="Sec23-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div>
                
                
                <div id="MOESM3"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM3_ESM.mp4">41592_2020_823_MOESM3_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Stretching of the unswollen 36% (wt/vol) entangled pAAm hydrogel shown in Extended Data Fig. 1c.</p></div></div>
                <div id="MOESM4"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM4_ESM.mp4">41592_2020_823_MOESM4_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Stretching of the ELASTicized 2-mm-thick coronal human brain hemisphere slab shown in Fig. 2c.</p></div></div>
                <div id="MOESM5"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM5_ESM.mp4">41592_2020_823_MOESM5_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Stretching of an ELASTicized 1-mm-thick coronal mouse brain hemisphere slice.</p></div></div>
                <div id="MOESM6"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM6_ESM.mp4">41592_2020_823_MOESM6_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Compression of ELASTicized mouse organs.</p></div></div>
                <div id="MOESM7"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM7_ESM.mp4">41592_2020_823_MOESM7_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>A demonstration of the operation of the cyclic compression device. The cycling was set to 1 mm and 40 s for compression and 10 mm and 10 s for release.</p></div></div>
                <div id="MOESM8"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM8_ESM.mp4">41592_2020_823_MOESM8_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Axial navigation of the image volumes of the ELASTicized 3-mm-thick mouse brain block in Fig. 3l showing preserved tissue micro-structures after about 1,500 times of repeated sixfold compression for 1 d. A representative image from four repeated experiments included in Fig. 3m.</p></div></div>
                <div id="MOESM9"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM9_ESM.mp4">41592_2020_823_MOESM9_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Axial navigation of the image volume in Fig. 3n showing full penetration and complete labeling of an ELASTicized 5-mm-thick human brain cortical block with anti-GFAP antibodies. Experiment was not repeated.</p></div></div>
                <div id="MOESM10"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM10_ESM.mp4">41592_2020_823_MOESM10_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Calretinin-positive interneurons (orange) and NF-L-positive axonal projections (blue) in the human cerebral cortex sample shown in Fig. 3o that was rapidly immunolabeled via sample-thinning. Experiment was repeated two times with similar results.</p></div></div>
                <div id="MOESM11"><h3 class="c-article__sub-heading"><a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM11_ESM.mp4">41592_2020_823_MOESM11_ESM.mp4</a></h3><div class="serif suppress-bottom-margin add-top-margin standard-space-below" data-test="bottom-caption"><p>Axial navigation of the image volume in Fig. 3o showing complete immunolabeling of calretinin-positive interneurons and NF-L-positive axonal projections. Experiment was repeated two times with similar results.</p></div></div>
              <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–8 and Tables 1–3.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM3_ESM.mp4" data-supp-info-image="">Supplementary Video 1</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Stretching of the unswollen 36% (wt/vol) entangled pAAm hydrogel shown in Extended Data Fig. 1c.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM4_ESM.mp4" data-supp-info-image="">Supplementary Video 2</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Stretching of the ELASTicized 2-mm-thick coronal human brain hemisphere slab shown in Fig. 2c.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM5_ESM.mp4" data-supp-info-image="">Supplementary Video 3</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Stretching of an ELASTicized 1-mm-thick coronal mouse brain hemisphere slice.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM6_ESM.mp4" data-supp-info-image="">Supplementary Video 4</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Compression of ELASTicized mouse organs.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM7_ESM.mp4" data-supp-info-image="">Supplementary Video 5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>A demonstration of the operation of the cyclic compression device. The cycling was set to 1 mm and 40 s for compression and 10 mm and 10 s for release.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM8_ESM.mp4" data-supp-info-image="">Supplementary Video 6</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Axial navigation of the image volumes of the ELASTicized 3-mm-thick mouse brain block in Fig. 3l showing preserved tissue micro-structures after about 1,500 times of repeated sixfold compression for 1 d. A representative image from four repeated experiments included in Fig. 3m.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM9_ESM.mp4" data-supp-info-image="">Supplementary Video 7</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Axial navigation of the image volume in Fig. 3n showing full penetration and complete labeling of an ELASTicized 5-mm-thick human brain cortical block with anti-GFAP antibodies. Experiment was not repeated.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM10_ESM.mp4" data-supp-info-image="">Supplementary Video 8</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Calretinin-positive interneurons (orange) and NF-L-positive axonal projections (blue) in the human cerebral cortex sample shown in Fig. 3o that was rapidly immunolabeled via sample-thinning. Experiment was repeated two times with similar results.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-020-0823-y/MediaObjects/41592_2020_823_MOESM11_ESM.mp4" data-supp-info-image="">Supplementary Video 9</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Axial navigation of the image volume in Fig. 3o showing complete immunolabeling of calretinin-positive interneurons and NF-L-positive axonal projections. Experiment was repeated two times with similar results.</p></div></div></div></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-category="article body" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Elasticizing%20tissues%20for%20reversible%20shape%20transformation%20and%20accelerated%20molecular%20labeling&amp;author=Taeyun%20Ku%20et%20al&amp;contentID=10.1038%2Fs41592-020-0823-y&amp;publication=1548-7091&amp;publicationDate=2020-05-18&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41592-020-0823-y" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41592-020-0823-y" data-track="click" data-track-action="Click Crossmark" data-track-category="article body" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" /></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Ku, T., Guan, W., Evans, N.B. <i>et al.</i> Elasticizing tissues for reversible shape transformation and accelerated molecular labeling.
                    <i>Nat Methods</i>  (2020). https://doi.org/10.1038/s41592-020-0823-y</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-category="article body" data-track-label="link" href="/articles/s41592-020-0823-y.ris">Download citation<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="u-clearfix c-bibliographic-information__value"><time datetime="2019-04-03">03 April 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="u-clearfix c-bibliographic-information__value"><time datetime="2020-04-01">01 April 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="u-clearfix c-bibliographic-information__value"><time datetime="2020-05-18">18 May 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--doi"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="u-clearfix c-bibliographic-information__value"><a href="https://doi.org/10.1038/s41592-020-0823-y" data-track="click" data-track-action="view doi" data-track-category="article body" data-track-label="link" itemprop="sameAs">https://doi.org/10.1038/s41592-020-0823-y</a></span></p></li></ul><div data-component="share-box"></div><div data-component="article-info-list"></div></div></div></div></div></section>

                        

                        

                        <span data-recommended="jobs"></span>
                    </div>
                </div>

                <div class="c-article-extras u-hide-print" role="complementary" data-container-type="reading-companion" data-track-component="reading companion">
                    <noscript>
                        
                            
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
            
            <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
            
    
        
        <div class="c-pdf-download u-clear-both">
            <a href="/articles/s41592-020-0823-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            </a>
        </div>
        
    

        
    
</div>

                        
                    </noscript>
                    
                        <div class="c-nature-box c-nature-box--side u-display-none" aria-hidden="true" data-component="entitlement-box" id=entitlement-box-desktop>
    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
    
        
        <div class="c-pdf-download u-clear-both">
            <a href="/articles/s41592-020-0823-y.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
                <span>Download PDF</span>
                <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
            </a>
        </div>
        
    

    
</div>

                    

                    
                        
                    

                    

                    <div class="c-reading-companion">
                        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
                            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                                
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-gpt-unitpath="/285/nmeth.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41592-020-0823-y;doi=10.1038/s41592-020-0823-y;subjmeta=1407,1555,1647,2225,245,378,631;kwrd=Fluorescence imaging,Histology,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nmeth.nature.com/article&amp;sz=300x250&amp;c=-145496458&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41592-020-0823-y%26doi%3D10.1038/s41592-020-0823-y%26subjmeta%3D1407,1555,1647,2225,245,378,631%26kwrd%3DFluorescence imaging,Histology,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nmeth.nature.com/article&amp;sz=300x250&amp;c=-145496458&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41592-020-0823-y%26doi%3D10.1038/s41592-020-0823-y%26subjmeta%3D1407,1555,1647,2225,245,378,631%26kwrd%3DFluorescence imaging,Histology,Neuroscience"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                            </div>
                            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
                            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
                        </div>
                    </div>
                </div>
            </article>
        </div>
    </div>
</div>


<div class="hide-print js-header-menu menu mb20 z-index-50 composite-layer background-white border-bottom-2 border-gray-medium tighten-line-height" id="menu" data-track-component="menu">
    <div class="menu-inner content js-hide">
        <nav>
            <div class="cleared border-bottom-1 border-gray-medium ml20 mr20">
                <div class="grid-ng grid-1of4 mq875-grid-12">
                    <h3 class="h3 pa20 mq875-pa0 mq875-mt20">
                        Nature Methods<span class="visually-hidden"> menu</span>
                    </h3>
                </div>
                
    
        
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <ul class="clean-list mb0">
                
                <li class="mb4 pr10"><a href="/nmeth/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link">News &amp; Comment</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/research" data-track="click" data-track-action="research" data-track-label="link">Research</a></li>
                
            </ul>
        </div>
        

        
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <ul class="clean-list mb0">
                
                <li class="mb4 pr10"><a href="/nmeth/current-issue" data-track="click" data-track-action="current issue" data-track-label="link">Current Issue</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/browse-issues" data-track="click" data-track-action="browse issues" data-track-label="link">Browse Issues</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/articles" data-track="click" data-track-action="browse articles" data-track-label="link">Browse Articles</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/collections" data-track="click" data-track-action="browse collections" data-track-label="link">Browse Collections</a></li>
                
            </ul>
        </div>
        

        
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <ul class="clean-list mb0">
                
                <li class="mb4 pr10"><a href="/nmeth/about" data-track="click" data-track-action="about the journal" data-track-label="link">About the Journal</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/for-authors" data-track="click" data-track-action="for authors" data-track-label="link">For Authors</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">For Reviewers</a></li>
                
                <li class="mb4 pr10"><a href="/nmeth/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">Web Feeds</a></li>
                
            </ul>
        </div>
        
    

            </div>
            <div class="cleared ml20 mr20">
                
    <div class="cleared pb10">
        <div class="grid-ng grid-1of4 mq875-grid-12">
            <h3 class="serif pa20 mq875-pa0 mq875-mt20">Nature Research<span class="visually-hidden"> menu</span></h3>
        </div>
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <h4 class="mb4">Our Journals</h4>
            <ul class="clean-list mb0">
                <li class="mb4 pr10"><a href="/nature" data-track="click" data-track-action="nature" data-track-label="link">Nature</a></li>
                <li class="mb4 pr10"><a href="/ncomms" data-track="click" data-track-action="nature communications" data-track-label="link">Nature Communications</a></li>
                <li class="mb4 pr10"><a href="/nprot" data-track="click" data-track-action="nature protocols" data-track-label="link">Nature Protocols</a></li>
                <li class="mb4 pr10"><a href="/srep" data-track="click" data-track-action="scientific reports" data-track-label="link">Scientific Reports</a></li>
                <li class="mb4 pr10"><a href="/siteindex" data-track="click" data-track-action="journal a-z view all" data-track-label="link">View all journals</a></li>
            </ul>
        </div>
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <h4 class="mb4">Subjects</h4>
            <ul class="clean-list mb0">
                <li class="mb4 pr10"><a href="/subjects/biological-sciences" data-track="click" data-track-action="biological sciences" data-track-label="link">Biological Sciences</a></li>
                <li class="mb4 pr10"><a href="/subjects/scientific-community-and-society" data-track="click" data-track-action="scientific community and society" data-track-label="link">Scientific Community &amp; Society</a></li>
                <li class="mb4 pr10"><a href="/subjects/earth-and-environmental-sciences" data-track="click" data-track-action="earth and environmental sciences" data-track-label="link">Earth &amp; Environmental Sciences</a></li>
                <li class="mb4 pr10"><a href="/subjects/health-sciences" data-track="click" data-track-action="health sciences" data-track-label="link">Health Sciences</a></li>
                <li class="mb4 pr10"><a href="/subjects/physical-sciences" data-track="click" data-track-action="physical sciences" data-track-label="link">Physical Sciences</a></li>
                <li class="mb4 pr10"><a href="/subjects" data-track="click" data-track-action="subjects view all" data-track-label="link">View all subjects</a></li>
            </ul>
        </div>
        <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
            <h4 class="mb4">More</h4>
            <ul class="clean-list mb0">
                <li class="mb4 pr10"><a href="https://support.nature.com/support/home" data-track="click" data-track-action="subscriptions" data-track-label="link">Contact us</a></li>
                <li class="mb4 pr10"><a href="/authors/index.html" data-track="click" data-track-action="authors and referees" data-track-label="link">Authors &amp; Referees</a></li>
                <li class="mb4 pr10"><a href="/libraries/index.html" data-track="click" data-track-action="librarians" data-track-label="link">Librarians</a></li>
                <li class="mb4 pr10"><a href="/advertising/index.html" data-track="click" data-track-action="advertisers" data-track-label="link">Advertisers</a></li>
                <li class="mb4 pr10"><a href="/npg_/press_room/index.html" data-track="click" data-track-action="press" data-track-label="link">Press</a></li>
                <li class="mb4 pr10"><a href="/npg_/index_npg.html" data-track="click" data-track-action="about" data-track-label="link">About Nature Research</a></li>
            </ul>
        </div>
    </div>



            </div>
        </nav>
    </div>
</div>



    <div id="search-menu" class="menu pt20 mb20 z-index-50 composite-layer background-white border-gray-medium border-bottom-2" data-component="tray" data-track-component="header">
        <div class="menu-inner js-hide">
            <section>
                <h2 class="visually-hidden">Search</h2>
                <div class="hide-print content mb40 mq1200-padded position-relative" data-test="inline-search">
                    <div class="pt30 cleared background-white sans-serif">
                        <div class="grid grid-8 grid-left-2 mq875-grid-12 mq875-ml0 mq640-pb30">
                            <form action="/search" method="get" role="search" autocomplete="off" class="standard-space-below" data-track="submit" data-track-action="search" data-track-label="form">
                                <label for="keywords" class="block strong">Article search</label>
                                <div class="position-relative">
                                    <input type="search" id="keywords" class="border-gray border-all-1 equalize-line-height pa10 pr40 box-sizing grid-12" name="q" value="" placeholder="Search by keywords or author" data-test="search-keywords">
                                    <div class="position-absolute position-right position-top mt1 mr1">
                                        <button type="submit" class="icon icon-center search-btn kill-border" data-test="search-submit">
                                            Search
                                        </button>
                                    </div>
                                    <p class="mb0 mt4 text14"><a href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">Advanced search</a></p>
                                </div>
                            </form>
                            <h3 class="h3 strong mb4 border-gray-medium border-bottom-1 sans-serif">Quick links</h3>
                            <ul class="clean-list mb0 text14">
                                <li class="grid-ng grid-1of2 mb6 mt6"><a href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
                                <li class="grid-ng grid-1of2 mb6 mt6"><a href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
                                <li class="grid-ng grid-1of2"><a href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
                                <li class="grid-ng grid-1of2"><a href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
            </section>
        </div>
    </div>




<footer role="contentinfo">
    
        
    


    

    <div itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        
            <div class="container-footer mt40 cleared container-footer-full">
                
                    <div class="content pt20 pr30 pl30">
                        <div class="grid grid-12 last clear-float cleared mb15 text-gray-light text13 flex-box flex-wrap-reverse">
                            <h2 aria-level="2" class="text13 strong emphasis ma0 sans-serif" itemprop="name">Nature Methods</h2>
                            <p class="pt0 pr10 pb0 pl10 ma0"><abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1548-7105</span> (online)</p>
                        </div>
                    </div>
                

                
    <div class="grid grid-12 pl30 pr30 last" id="footer">
        <h2 aria-level="2" class="hide">nature.com sitemap</h2>
    </div>

    <div class="pl30 pr30 content" data-track-component="footer">

        <div class="grid grid-12 last clear-float cleared mb5 pt20 text-gray-light flex-wrap">
            <div class="grid grid-3 mq875-grid-12 just-mq640-last just-mq875-last mb20"><img alt="Nature Research" src="/static/images/natureresearch-logo.2fe351a520.svg" loading="lazy" width="255" height="33"></div>
            <ul class="grid grid-7 mq875-grid-8 mq640-grid-12 just-mq640-last clean-list mb10 mt10 text15 u-hide-print">
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" class="text-gray-light" data-track-label="link">About us</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq480-last"><a href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link" class="text-gray-light">Press releases</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link" class="text-gray-light">Press office</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 last"><a href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" class="text-gray-light" data-track-label="link">Contact us</a></li>
            </ul>
            <ul class="grid grid-2 mq875-grid-4 mq640-grid-12 last clean-list mb10 mt4 u-hide-print">
                <li class="grid grid-4 mq875-grid-3"><a href="https://www.facebook.com/nature/" data-track="click" data-track-action="facebook" data-track-label="link" class="text-gray-light"><img src="/static/images/fb.49ba391f15.svg" class="cleared text-footer-img" alt="Facebook"/></a></li>
                <li class="grid grid-4 mq875-grid-3"><a href="https://twitter.com/nresearchnews?lang=en" data-track="click" data-track-action="twitter" data-track-label="link" class="text-gray-light"><img src="/static/images/twitter.a90dbeb03a.svg" class="cleared text-footer-img" alt="Twitter"/></a></li>
                <li class="grid grid-4 mq875-grid-3 last"><a href="https://www.youtube.com/channel/UCvCLdSgYdSTpWcOgEJgi-ng" data-track="click" data-track-action="youtube" data-track-label="link" class="text-gray-light"><img src="/static/images/youtube.6071f6012c.svg" class="cleared text-footer-img" alt="Youtube"/></a></li>
            </ul>
        </div>

        <div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-bottom hide-print u-hide-print">
            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Discover content</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link" class="text-gray-light">Journals A-Z</a></li>
                    <li class="pb4"><a href="https://www.nature.com/subjects/" data-track="click" data-track-action="article by subject" data-track-label="link" class="text-gray-light">Articles by subject</a></li>
                    <li class="pb4"><a href="https://nano.nature.com/" data-track="click" data-track-action="nano" data-track-label="link" class="text-gray-light">Nano</a></li>
                    <li class="pb4"><a href="https://www.nature.com/protocolexchange/" data-track="click" data-track-action="protocol exchange" data-track-label="link" class="text-gray-light">Protocol Exchange</a></li>
                    <li class="pb4"><a href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link" class="text-gray-light">Nature Index</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last">
                <h3 class="text-footer-heading">Publish with us</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/authors/author_resources/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link" class="text-gray-light" >Guide to Authors</a></li>
                    <li class="pb4"><a href="https://www.nature.com/authors/peer_review/" data-track="click" data-track-action="guide to referees" data-track-label="link" class="text-gray-light">Guide to Referees</a></li>
                    <li class="pb4"><a href="https://www.nature.com/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link" class="text-gray-light">Editorial policies</a></li>
                    <li class="pb4"><a href="http://www.nature.com/openresearch/publishing-with-npg/" data-track="click" data-track-action="open access" data-track-label="link" class="text-gray-light">Open access</a></li>
                    <li class="pb4" ><a href="https://www.nature.com/reprints/" data-track="click" data-track-action="reprints and permissions" data-track-label="link" class="text-gray-light">Reprints &amp; permissions</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-last">
                <h3 class="text-footer-heading">Researcher services</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link" class="text-gray-light">Research data</a></li>
                    <li class="pb4"><a href="https://authorservices.springernature.com/go/nr" data-track="click" data-track-action="language editing" data-track-label="link" class="text-gray-light">Language editing</a></li>
                    <li class="pb4"><a href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link" class="text-gray-light">Scientific editing</a></li>
                    <li class="pb4"><a href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link" class="text-gray-light">Nature Masterclasses</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/product/researcher-training/" data-track="click" data-track-action="nature research academies" data-track-label="link" class="text-gray-light">Nature Research Academies</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-clear just-mq640-last full-size-last just-mq1200-last just-mq875-last just-mq480-last">
                <h3 class="text-footer-heading">Libraries &amp; institutions</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link" class="text-gray-light">Librarian service &amp; tools</a></li>
                    <li class="pb4"><a href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link" class="text-gray-light">Librarian portal</a></li>
                    <li class="pb4"><a href="http://www.nature.com/openresearch/about-open-access/information-for-institutions/" data-track="click" data-track-action="open research" data-track-label="link" class="text-gray-light">Open research</a></li>
                </ul>
            </div>
        </div>

        <div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-top hide-print u-hide-print">
            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Advertising &amp; partnerships</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link" class="text-gray-light">Advertising</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link" class="text-gray-light">Partnerships &amp; Services</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link" class="text-gray-light">Media kits</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link" class="text-gray-light">Branded content</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq875-last">
                <h3 class="text-footer-heading">Career development</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/naturecareers" data-track="click" data-track-action="nature careers" data-track-label="link" class="text-gray-light">Nature Careers</a></li>
                    <li class="pb4"><a href="https://www.nature.com/natureconferences/" data-track="click" data-track-action="nature conferences" data-track-label="link" class="text-gray-light">Nature<span class="visually-hidden"> </span> Conferences</a></li>
                    <li class="pb4"><a href="https://www.nature.com/natureevents/" data-track="click" data-track-action="nature events" data-track-label="link" class="text-gray-light">Nature<span class="visually-hidden"> </span> events</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Regional websites</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link" class="text-gray-light">Nature China</a></li>
                    <li class="pb4"><a href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link" class="text-gray-light">Nature India</a></li>
                    <li class="pb4"><a href="https://www.natureasia.com/ja-jp/" data-track="click" data-track-action="nature japan" data-track-label="link" class="text-gray-light">Nature Japan</a></li>
                    <li class="pb4"><a href="https://www.natureasia.com/ko-kr/" data-track="click" data-track-action="nature korea" data-track-label="link" class="text-gray-light">Nature Korea</a></li>
                    <li class="pb4"><a href="https://www.nature.com/nmiddleeast/" data-track="click" data-track-action="nature middle east" data-track-label="link" class="text-gray-light">Nature Middle East</a></li>
                </ul>
            </div>
        </div>
    </div>


            </div>
        
    </div>

    
    <div class="container-footer-box cleared text14 border-gray-medium border-top-1">
        <div class="content">
            <img src="/static/images/sn-logo.4368f3b177.svg" alt="Springer Nature" loading="lazy" width="140" height="14" class="cleared"/>
            <p class="mb10 text-gray-light">© 2020 Springer Nature Limited</p>
            <ul class="ma0 clean-list grid grid-12 hide-print u-hide-print">
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a href="https://www.nature.com/info/privacy.html" data-track="click" data-track-action="privacy policy" data-track-label="link" class="text-gray-light">Privacy Policy</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a href="https://www.nature.com/info/cookies.html" data-track="click" data-track-action="use of cookies" data-track-label="link" class="text-gray-light">Use of cookies</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq875-last just-mq480-last"><a href="javascript:;" class="optanon-toggle-display text-gray-light" data-track="click" data-track-action="manage cookies" data-track-label="link">Manage cookies</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a href="https://www.nature.com/info/legal_notice.html" data-track="click" data-track-action="legal notice" data-track-label="link" class="text-gray-light">Legal notice</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a href="https://www.nature.com/info/accessibility_statement.html" data-track="click" data-track-action="accessibility statement" data-track-label="link" class="text-gray-light">Accessibility statement</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 last"><a href="https://www.nature.com/info/tandc.html" data-track="click" data-track-action="terms and conditions" data-track-label="link" class="text-gray-light">Terms &amp; Conditions</a></li>
            </ul>
        </div>
    </div>



    
    <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
            <path d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
            <path d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
            <path d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
            <path d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 14 14">
            <path d="M13.545 12.648a.641.641 0 01.006.903.646.646 0 01-.903-.006l-2.664-2.663a6.125 6.125 0 11.897-.898l2.664 2.664zm-7.42-1.273a5.25 5.25 0 100-10.5 5.25 5.25 0 000 10.5z"></path>
        </symbol>
    </svg>

</footer>




    
        <div class="c-site-messages message hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant sans-serif"
    data-component-id="nature-briefing-banner"
    data-component-expirydays="30"
    data-component-trigger-scroll-percentage="15"
    data-track="in-view"
    data-track-action="in-view"
    data-track-category="nature briefing"
    data-track-label="banner visible">

    <div class="">

        
        <div class="c-site-messages__banner-large">

            <div class="c-site-messages__close-container">
                <button class="c-site-messages__close"
                    data-track="click"
                    data-track-category="nature briefing"
                    data-track-label="banner dismiss">
                    <?xml version="1.0" encoding="UTF-8" standalone="no"?>
                    <svg width="24px" height="24px" focusable="false" aria-hidden="true" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
                        <title>Close banner</title>
                        <defs></defs>
                        <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                            <rect opacity="0" x="0" y="0" width="24" height="24"></rect>
                            <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#006699"></path>
                        </g>
                    </svg>
                    <span class="visually-hidden">Close</span>
                </button>
            </div>

            <div class="c-site-messages__form-container">
                <form action="/briefing/signup/formfeedback" method="post" class="nature-briefing-banner__form box-sizing grid grid-12 last" data-location="banner" data-track="submit" data-track-action="transmit-form">
                    <div class="grid grid-6">
                        <div class="text13 grid grid-7">
                            <span class="block strong extra-tight-line-height sans-serif">
                                
                                Sign up for the <em>Nature Briefing</em> newsletter for a daily update on COVID-19 science.
                                
                            </span>
                        </div>

                        <div class="grid grid-5 last">
                            <input id="briefing-box-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="DirectEmailBannerCovid19">
                            <input id="briefing-box-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">
                            <label for="banner-EmailAddressInput" class="visually-hidden">Enter your email address</label>
                            <input class="nature-briefing-banner__email-input border-gray border-all-1 equalize-line-height pa10 box-sizing text13" type="email" id="banner-EmailAddressInput" name="email" value="" placeholder="Enter your email address" required="true" aria-required="true">
                        </div>
                    </div>

                    <div class="grid grid-6 last">
                        <div class="grid grid-9">
                            <div class="grid grid-12 last">
                                <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="1" required>
                                <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                            </div>
                        </div>

                        <div class="grid grid-3 last">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>
                    </div>
                </form>
            </div>

        </div>

        
        <div class="c-site-messages__banner-small">

            <div class="c-site-messages__content text14">
                <span class="strong serif">Get the most important science stories of the day, free in your inbox.</span>
                <a class="nature-briefing__link text14 sans-serif"
                    data-track="click"
                    data-track-category="nature briefing"
                    data-track-label="banner CTA to site"
                    target="_blank" rel="noreferrer noopener"
                    href="/briefing/signup/?origin=Nature&amp;originReferralPoint=EmailBanner">Sign up for Nature Briefing
                </a>
            </div>
            <div class="c-site-messages__close-container pin-right">
                <button class="c-site-messages__close"
                    data-track="click"
                    data-track-category="nature briefing"
                    data-track-label="banner dismiss">
                    <span class="icon--inline inline-block"><?xml version="1.0" encoding="UTF-8" standalone="no"?>
                        <svg width="24px" height="24px" focusable="false" aria-hidden="true" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
                            <title>Close banner</title>
                            <defs></defs>
                            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                                <rect opacity="0" x="0" y="0" width="24" height="24"></rect>
                                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#006699"></path>
                            </g>
                        </svg>
                    </span>
                    <span class="visually-hidden">Close</span>
                </button>
            </div>

        </div>

    </div>

</div>

    






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41592-020-0823-y&amp;format=js&amp;last_modified=2020-05-18" async></script>
<img src="/platform/track/article/s41592-020-0823-y" width="1" height="1" alt="" class="visually-hidden"/>

</body>
</html>
